---
layout: page 
title: Cnidofest 2022 Full Program
permalink: /cnidofest-2022/program
---


<span style= "color:red">**All events except dinner are at the UC Davis Convention Center**</span>

- table of contents
{:toc}

## Schedule

*Please note:* 	*There will be no dinner served on Wednesday. We recommend you eat before* *arriving to register.* 

### Wednesday, September 7

| Time           | Event                                                        |
| -------------- | -------------------------------------------------------------|
| 6:30 pm        | Registration and Check-in (UC Davis Conference Center)       |
| 7:30 pm        | Welcome                                                      |
| 7:45 pm        | **Session 1: Genomics**<br>*Session Chair: James Gahan*      |
|                | Jack Cazet: “New *Hydra* genomes reveal conserved principles of hydrozoan transcriptional regulation” (p. 16)<br/><br/>Sally Chang: “Insights from the draft genome assembly for the hydrozoan *Podocoryna carnea*: Just the tip of the tentacle” (p. 17)<br/><br/>Tetsuo Kon: “Genomic divergence and dynamics of the three stem cell lineages in *Hydra*” (p. 18)<br/><br/>Karly Higgins-Poling: “Assessing evolutionary history of symbiotic Scyphozoa: A case study, *Mastigias papua* in the marine lakes of Palau, and beyond” (p. 19) |
| 9:00 pm        | 🍻 Know About Each Other in the Bar<br/>(Downtown Davis on G Street) |


### Thursday, September 8 

| Time | Event|
| ------------------------- | ------------------------------------------------------------ |
| 8:00 am            | 🥐 Light Continental Breakfast                                  |
| 8:45 am           | **Session 2: Evolution & Regeneration**<br/>*Session Chair: David Gold* |
|                           | Anna Klompen: “Division of (toxic) labor: Venom distribution and utility across distinct polyp types of *Hydractinia symbiolongicarpus*” (p. 21)<br/><br/>Alonso Delgado: “Sea anemone Insulin-like peptides, For all time. Always.” (p. 22)<br/><br/>Áine Varley: “The developmental potential of a single i-cell in *Hydractinia*”	 (p. 23) |
| 10:00 am          | ☕️ Coffee Break                                                 |
| 10:30 pm          | **Session 3: Development I**<br/>*Session Chair: Celina Juliano* |
|                           | Diego Calderon, Invited Technology Speaker: “The continuum of *Dro-**sophila* embryonic development at single cell resolution”(p. 24)<br/><br/>Layla Al-Shaer: “Population density modulates asexual reproduction & gene expression in *Nematostella vectensis*” (p. 25)<br/><br/>Charisious Tsiairis: The transcription factor Zic4 prevents epithelial transdifferentiation in *Hydra*” (p. 26)|
| 12:00 pm           | 🥪 Box Lunch                                                    |
| 1:15 pm            | **Session 4: Neurobiology**<br>*Session Chair: Brady Weissbourd* |
|                           | Abby Primack: “Characterizing neurogenesis pathways in *Hydra vulgaris*” (p. 28) <br/> <br/>Minghe Cheng: “Investigating the role of NvashA during *Nematostella* Neurogenesis” (p. 29)<br/> <br/>Wataru Yamamoto: “Studying neuronal mechanisms of somersaulting in *Hydra vulgaris*” (p. 30)<br/> <br/>Kelly Kim: “Phototaxis is a state-dependent complex behavior in *Hydra* *vulgaris*” (p. 31) |
| 2:30 pm            | ☕️ Coffee Break                                                 |
| 3:00 pm            | **Session 5: Evo Devo I** <br/>*Session Chair: Paulyn Cartwright* |
|                           | Gaku Kumano: “Branching morphogenesis of *Cladonema* medusa tentacles” (p. 32)<br/> <br/>Yareli Alvarez: “Inducible organs of aggression in the plumose sea anemone, *Metridium senile*, as a model for the evolution of cell type plasticity” (p. 33)<br/> <br/>Julia Baranyk: “Cnidarian-specific neuropeptide RPamide decelerates the timing of life cycle transition in the sea anemone *Nematostella vectensis*” (p. 34)<br/> <br/>Alison Cole: “A cell-type atlas from a scyphozoan jellyfish *Aurelia* sp. provides insights into cell-type diversity and reveals conserved cell specification mechanisms across the Cnidaria” (p. 35) |
| 4:15 pm            | ☕️ Coffee Break                                                 |
| 4:45 pm            | 🌩 **Lightning Talks**                                              |
|                           | Zeeshan Banday, <br/>Natascha Bartsch,  <br/>JK Da-Anoy, <br/>Iris Juanico,  <br/>Grace Snyder,<br/>Qingru Xu. |
| 5:30 pm            | **Poster Session 1** |
|  | 1 - Namrata Ahuja (p. 68) <br/>3 - Zeeshan Banday (p. 70) <br/>5 - Natascha Bartsch (p. 72) <br/>7 - Jun Cai (p. 74) <br/>9 - JK Da-Anoy (p. 76) <br/>11 - Alondra Escobar (p. 78) <br/>13 - Stuart Jaeger (p. 80) <br/>15 - Iris Juanico (p. 82) <br/>17 - Whitney Leach (p. 84) <br/>19 - Jocelyn Malamy (p. 86) <br/>21 - Cody Miner (p. 88) <br/>23 - Clara Nuninger (p. 90) <br/>25 - Kari Price (p. 92) <br/>27 - Grace Snyder (p. 94) <br/>29 - Marina Stoilova (p. 96) <br/>31 - Justin Waletich (p. 98) <br/>33 - Qingru Xu (p. 100) <br/>35 - Mengjin Zhang (p. 102) <br/>37 - Bruno Gideon Bergheim (p. 46) <br/>39 - Sosuke Fujita (p. 64) |
| 7:30 pm            | 🍽 Dinner with Cash Bar (Alumni Center)                         |
| 9:30 pm                   | 🍻 Know About Each Other in the Bar <br/>(Downtown Davis on G Street) |

### Friday, September 9

| Time |                                                              Event|
| ----------------------- | ------------------------------------------------------------ |
| 8:00 am          | 🥐 Light Continental Breakfast                                  |
| 8:45 am         | **Session 6: Physiology I** <br/>*Session Chair: Patrick Steinmetz* |
|                         | Allyson DeMerlis: “Early-hour molecular responses of wound healing in *Pocillopora damicornis*” (p. 36)<br/> <br/>Samuel Bedgood: “Algal symbiont presence and not photosynthetic contribution affects the development of *Exaiptasia diaphana* asexual offspring” (p. 37)<br/> <br/>Cory Berger: “How does *Nematostella’s* circadian clock behave under conflicting light and temperature cycles?” (p. 38)<br/> <br/>Olivier Cochet-Escartin: “Mechanical characterization of regenerating*Hydra vulgaris* tissue pieces” (p. 39) |
| 10:00 am        | ☕️ Coffee Break                                                 |
| 10:30 pm        | **Session 7: Physiology II** <br/>*Session Chair: Matt Nicotra* |
|                         | James Letts, Invited Technology Speaker: “New approaches to structural biology using single particle cryoEM” (p. 40)<br/> <br/>Kathrin Garschall: “Nutritional regulation of growth in the sea anemone *Nematostella vectensis*” (p. 41)<br/> <br/>Lys Isma: “Saved by the Cell: Immune cells and Stony Coral Tissue Loss Disease (SCTLD)” (p. 42)<br/> <br/>Yehu Moran: “The cnidarian antiviral immune system reflects ancestral complexity” (p. 43) |
| 12:00 pm         | 🥪 Box Lunch                                                 |
| 1:15 pm          | **Session 8: Cell Biology I** <br/>*Session Chair: Leslie Babonis* |
|                         | Manuel Michaca: “Visualizing the expression of Alr1 in *Hydractinia symbiolongicarpus*” (p. 44)<br/> <br/>Anush Kosakyan: “Parasitic strategies in Cnidaria from omics perspective. Case study myxozoan parasite *Sphaerspora molnari*” (p. 45)<br/> <br/>Bruno Gideon Bergheim: “Extracellular matrix dynamics during the life history of *Nematostella vectensis*” (p. 46) <br/> <br/>Bert Hobmayer: “A *Hydra* Claudin cell-cell contact protein is involved in epithelial tissue dynamics, regeneration, and osmoregulation”(p. 47) |
| 2:30 pm          | ☕️ Coffee Break                                                 |
| 3:00 pm          | **Session 9: Cell Biology II** <br/>*Session Chair: Anush Kosakyan* |
|                         | Marion Lechable: “Evolutionary dynamics of the stem cell factor Myc and functional conservation of the unique *Hydra* Myc3 protein” (p. 48)<br/> <br/>Maria Valadez Ingersoll: “Interactions between symbiosis, innate immunity, and nutrition” in cnidarians” (p. 49)<br/> <br/>Helen Horkan: “*Hydractinia* Genome stability” (p. 50)<br/> <br/>Tsuyoshi Momose: “Origin of metazoan and origin of jellyfish; genetic and cell-biology approaches.” (p. 51) |
| 4:15 pm          | ☕️ Coffee Break                                                 |
| 4:45 pm          | 🌩 **Lightning Talks**                                          |
|                         | Ndotimi Apulu,  <br/>Yamaly Barragan,  <br/>Wiebke Ehrlich,  <br/>Elizabeth Lee,  <br/>Hannah Morris Little,  <br/>Jingwei Song,  <br/> Amanda Yeo |
| 5:30 pm          | **Poster Session 2**                                         |
|                         | 2 - Ndotimi Apulu (p. 69) <br/>4 - Yamaly Barragan (p. 71) <br/>6 - Charlotte Benedict (p. 73) <br/>8 - Craig Ciampa (p. 75) <br/>10 - Wiebke Ehrlich (p. 77) <br/>12 - Addie Harrison (p. 79) <br/>14 - Dhiraj Jain (p. 81) <br/>16 - Hannah Justin (p. 83) <br/>18 - Elizabeth Lee (p. 85) <br/>20 - Kamille Maningding (p. 87) <br/>22 - Hannah Morse Little (p. 89) <br/>24 - Isabel Pen (p. 91) <br/>26 - Auston Rutledge (p. 93) <br/>28 - Jingwei Song (p. 95) <br/>30 - Aliyah True (p. 97) <br/>32 - Erick White (p. 99) <br/>34 - Amanda Yeo (p. 101) <br/>36 - Julia Baranyk (p. 34) <br/>38 - Jack Cazet (p. 103) <br/>40 - Abby Primack (p. 28) |
| 7:30 pm          | Dinner with Cash Bar (Alumni Center)                         |
| 9:30 pm                 | 🍻 Know About Each Other in the Bar<br/>(Downtown Davis on G Street) |

### Saturday, September 10

| Time |Event                                                              |
| -------------------------- | ------------------------------------------------------------ |
| 8:00 am             | Light Continental Breakfast                                  |
| 8:45 am            | **Keynote**<br/>                                             |
|                            | Mansi Srivastava: “Regeneration and development: the acoelomorph perspective” (p. 52) |
| 10:00 am           | ☕️ Coffee Break                                                 |
| 10:30 pm           | **Session 10: Development II**<br/>*Session Chair: Chiara Sinigaglia* |
|                            | Paula Miramon-Puertolas: “A post-larval stem-like cell population contributes to germinal and somatic lineages in the sea anemone *Nematostella vectensis*” (p. 53) <br/><br/>Fredrik Hugosson: “Functional characterization of canonical Wnt receptors and ligands required for oral-aboral patterning in *Nematostella vectensis*” (p. 54)<br/><br/>Keith Sabin: “An FGF activity gradient signals through a cnidarian paired-like homeodomain gene to direct apical sensory organ development in *Nematostella vectensis*” (p. 55)<br/><br/>Kerstin Ohler: “FGF signaling in the freshwater polyp *Hydra*” (p. 56)<br/><br/>Masha Brooun: “The Hippo pathway and *Hydra* morphogenesis.” (p. 57) |
| 12:00 pm            | 🥪 Box Lunch                                                    |
| 1:15 pm             | **Session 11: Evo Devo II**<br/>*Session Chair: Nagayasu Nakanishi* |
|                            | Lucas Leclére: “Genomic consequences of the loss of the polyp stage in the *scyphozoan Pelagia*” (p. 58)<br/><br/>Maciej Manko: “Origins of individuality in colonial siphonophores” (p. 59)<br/><br/>Mathew Travert: “Coevolution of the Tlx homeobox gene with medusa development (Cnidaria: Medusozoa)” (p. 60)<br/><br/>Patrick Steinmetz: “Yolk formation in a sea anemone provides insights into the evolution of animal nutrient transport” (p. 61) |
| 2:30 pm             | ☕️ Coffee Break                                                 |
| 3:00 pm             | **Session 12: Regeneration**<br/>*Session Chair: Christy Schnitzler* |
|                            | Anupama Hemalatha, Invited Technology Talk: “Live-imaging metabolic states in skin stem cells to reveal adaptations to oncogenic mutations” (p. 62) <br/><br/>Ben Cox: “Progenitor cell invasion during *Hydra vulgaris* head Regeneration” (p. 63) <br/><br/>Sosuke Fujita: “Distinct stem-like cell populations coordinate tissue elon-gation and regeneration in *Cladonema* medusa tentacles” (p. 64)<br/><br/>Sergio Campos: “Characterizing defects in Hydra oligactis foot Regeneration” (p. 65)<br/><br/>Chiara Sinigaglia: “To regenerate or not to regenerate? Towards an integrative model for the recovery of shapes in damaged *Clytia medusae*” (p. 66)<br/><br/>Miguel Salinas-Saavedra: “Senescence-induced cellular reprogramming drives whole-body regeneration” (p. 67) |
| 4:45 pm            | 🏆 Awards Ceremony & Meeting Wrap Up                             |
| 5:15 pm            | 📸 Group Photo                                                   |
| 5:45 pm            | 🏊‍♀️ Free Time 🥾 🌲 📕                                             |
| 7:30 pm            | 🍽 Dinner with Cash Bar (Alumni Center)                          |
| 10:00 pm           | 🍻 Know About Each Other in the Bar (Downtown Davis on G Street) |






## Speaker Abstracts 

### Wednesday, September 7

#### Session 1: Genomics

<p><strong>New </strong><strong><em>Hydra</em></strong><strong> New <i>Hydra</i> genomes reveal conserved principles of hydrozoan transcriptional regulation</strong></p>
<p><span style="text-decoration: underline;">Jack F. Cazet</span>1, Stefan Siebert2, Hannah Morris Little1, Philip Bertemes3, Abby S. Primack1, Peter Ladurner3, Matthias Achrainer3, Mark T. Fredriksen4, R. Travis Moreland4, Sumeeta Singh4, Suiyuan Zhang4, Tyra G. Wolfsberg4, Christine E. Schnitzler5, Andreas D. Baxevanis4, Oleg Simakov6, Bert Hobmayer3, and Celina E. Juliano1</p>
<p>1University of California, Davis, CA, USA</p>
<p>2Lyell Immunopharma, South San Francisco, CA, USA</p>
<p>3University of Innsbruck, Innsbruck, Austria</p>
<p>4National Institutes of Health, Bethesda, MD, USA</p>
<p>5University of Florida, St. Augustine, FL, USA</p>
<p>6University of Vienna, Vienna, Austria</p>
<p>The epithelial and interstitial stem cells of the freshwater polyp <em>Hydra</em> are the best characterized stem cell systems in any cnidarian, providing valuable insight into cell type evolution and the origin of stemness in animals. However, little is known about the transcriptional regulatory mechanisms that determine how these stem cells are maintained and how they give rise to their diverse differentiated progeny. To address such questions, a thorough understanding of transcriptional regulation in <em>Hydra</em> is needed. To this end, we generated extensive new resources for characterizing transcriptional regulation in <em>Hydra</em>, including new genome assemblies for <em>Hydra oligactis</em> and the AEP strain of <em>Hydra vulgaris</em>, an updated whole-animal single-cell RNA-seq atlas, and genome-wide maps of chromatin interactions, chromatin accessibility, sequence conservation, and histone modifications. These data revealed the existence of large chromatin interaction domains in the <em>Hydra</em> genome that likely influence transcriptional regulation in a manner distinct from topologically associating domains in bilaterians. We also uncovered the transcriptomic profiles of two previously molecularly uncharacterized cell types, isorhiza-containing nematocytes and somatic gonad ectoderm. We identified novel candidate regulators of cell-type-specific transcription, several of which have likely been conserved at least since the divergence of <em>Hydra</em> and the jellyfish <em>Clytia hemisphaerica</em> over 200 million years ago.</p>
<hr /><p><strong>Insights from the draft genome assembly for the hydrozoan </strong><strong><em>Podocoryna carnea</em></strong><strong>: Just the tip of the tentacle</strong></p>
<p><span style="text-decoration: underline;">E. Sally Chang</span><span style="text-decoration: underline;">1</span>, Matthew Travert2, Steven M. Sanders3, Anna M.L. Klompen2, Paul Gonzalez1, Sofia N. Barreira1, Shaniya Markalanda, Paulyn Cartwright2, Andreas D. Baxevanis1</p>
<p>1National Human Genome Research Institute</p>
<p>2University of Kansas</p>
<p>3University of Pittsburgh</p>
<p>4Johns Hopkins University Bloomberg School of Public Health.&nbsp;</p>
<p>Cnidarians are an excellent system for studying the evolution of complexity and novelty given their diversity in body plan organization and life history strategy. In particular, the hydrozoan family Hydractiniidae contains the entire spectrum of medusae development and truncation, making it particularly suitable for comparative genomics. Building on the genome sequencing of <em>Hydractinia</em>, with highly truncated medusa development, we are generating a high-quality genome sequence for the closely related hydrozoan <em>Podocoryna carnea</em>, which produces a pelagic medusa. These data will significantly advance comparative genomics studies aimed at identifying the genomic toolkit specific to production of the medusa life cycle stage and its constituent cell types, some of which are stage-specific and potentially convergent to bilaterian cell types. Our initial sequencing of this species revealed high heterozygosity, necessitating a variety of sequencing and bioinformatic strategies to successfully assemble a highly contiguous and complete draft genome sequence. In this presentation, we will discuss these efforts and the developmental and evolutionary insights this draft assembly has already provided. Additionally, we will present initial characterization of long-read, isoform-level RNAseq from male medusa- and female polyp-specific libraries. Ongoing improvements to the <em>P. carnea</em> genome sequence as well as additional planned RNAseq experiments will advance the entire <em>Podocoryna</em>+<em>Hydractinia</em> system as an important model for evolutionary genomics.</p>
<hr /><p><strong>Genomic divergence and dynamics of the three stem cell lineages in </strong><strong><em>Hydra</em></strong></p>
<p>Tetsuo Kon1, Koto Kon-Nanjo1, Tracy Chih-Ting Koubkova Yu2, Daniel E. Mart&iacute;nez3, Thomas Holstein2, Oleg Simakov1</p>
<p>1University of Vienna, Austria</p>
<p>2University of Heidelberg, Germany</p>
<p>3Pomona College, USA</p>
<p><em>Hydra vulgaris</em> is a fresh-water cnidarian with an exceptional regeneration capacity and lack of aging. These features are achieved by the activities of the three types of stem cells: ectodermal stem cell, endodermal stem cell, and interstitial stem cell. The <em>Hydra</em> genome consists of more than 60% repetitive elements (including transposons) which contributed to its increased genome size. Some transposons were shown to be activated early in regeneration, suggesting their potential roles in creating "dynamic" stem cell genomes. However, which transposable elements are activated in each stem cell type and how they influence the genomic architecture still remain to be investigated. Here we report our on-going efforts to reveal the genomic divergence and dynamics of the three stem cell types. First, in order to sequence the somatic genomes in each stem cell lineage, we are performing fluorescence-activated cell sorting of dissociated cells from the ectodermal actin::GFP/endodermal actin::RFP transgenic strain. We have obtained GFP-positive ectodermal cell lineage and RFP-positive endodermal cell lineage. The GFP/RFP-double negative cells have been collected as the interstitial stem cells and their derivatives. Our preliminary analyses using Nanopore sequencing reveals a characteristic set of structural variants, including transposable elements in the background population. These variants are compared to the spectrum observed in preliminary genomic data after dissociation-reaggregation. We corroborate our genomic data using publicly available single-cell RNA-seq data of <em>Hydra</em> and conducted analyses to identify cell type specific expression of transposable elements. Our work will help reveal the insertion hotspots of transposable elements in each of the three types of stem cells and dissect the regulatory roles of transposable elements in the stem cells both during evolution and during stress response.</p>
<hr /><p><strong>Assessing evolutionary history of symbiotic Scyphozoa: A case study, </strong><strong><em>Mastigias papua</em></strong><strong> in the marine lakes of Palau, and beyond</strong></p>
<p>Karly Higgins-Poling1, Michael Dawson1</p>
<p>1University of California, Merced</p>
<p>The evolutionary history of a species can be shrouded by time, and uncertainty about past events may be increased if considering additional species&rsquo; roles; these challenges combine in the complex interactions of symbiotic organisms. Advances in sequencing technologies offer increased resolution but also many gene trees with differing histories. Therefore, finding case studies that simplify these problems can provide insight. <em>Mastigias papua</em> populations in the marine lakes of Palau are symbiotic and aposymbiotic, offering a diversity of phenotypes-morphological and behavioral-in an unique system for studying evolution on relatively observable timescales (~12k years). To better apply sequencing technologies to these populations we used PacBio sequencing and assembled the 417.6Mb reference genome for <em>M. papua</em> at 19X coverage. This genome has been aligned to whole genome sequences to determine the degree of genetic differentiation among populations of <em>M. papua</em> with preliminary results suggesting clustering of genotype by habitat. The genome was also incorporated in designing baits (3932 (host) and 2225 (symbiont) for 455 (host) and 318 (symbiont) loci) for ultra-conserved elements to assess coevolution of Scyphozoa and their endosymbiotic Symbiodiniaceae. We are using these data to provide insight into population adaptation to multiple habitat types and changing climates, a behavioral, genomic and transcriptomic comparison between aposymbiotic and symbiotic populations, and an assessment of coevolution between hosts and symbionts.</p>

### Thursday, September 8

#### Session 2: Evolution & Regeneration

<hr /><p><strong>Division of (toxic) labor: Venom distribution and utility across distinct polyp types of </strong><strong><em>Hydractinia symbiolongicarpus</em></strong></p>
<p><span style="text-decoration: underline;">Anna Klompen</span>1, Paulyn Cartwright1</p>
<p>1Department of Ecology and Evolutionary Biology, University of Kansas, Kansas, USA</p>
<p>The phylum Cnidaria (jellyfish, hydroids, sea anemones, and corals) is one of the earliest diverging venomous animal groups, deploying their venoms through the discharge of highly pressurized secretary products called nematocysts. Nematocysts (housed in nematocytes) are produced continually over the lifetime of the organism and different morphological types are often unevenly distributed across functionally distinct tissues, lending to the prediction that different nematocyst types contain specific venom components tailored to specific ecological interactions. Differential gene expression of venom-like genes has been shown across the distinct tissues with different nematocyst types in sea anemones, but it remains unknown if colonial cnidarians partition their venoms according to function. To explore the relationship between venom production, nematocyst type specificity and ecological use in a colonial cnidarian we utilized the hydrozoan <em>Hydractinia symbiolongicarpus,</em> which displays a division of labor through functionally-specific polyps: gastrozooids (feeding), gonozooids (reproductive), and dactylozooids (predatory). We found that nematocyst types are unevenly distributed between these polyp types. Furthermore, using FAC-sorted nematocytes from a nematocyst-specific transgenic reporter line, we found several venom-like genes show differential gene expression between polyp types. Interestingly, members of a highly potent venom family known as jellyfish toxins (JFTs), which are often dominant in medically-relevant box jellyfish venoms, are differentially expressed between polyp types. We are characterizing spatial expression patterns of multiple JFTs across the <em>Hydractinia </em>colony to gain additional insight into the specific nematocyst types that are associated with these toxins. Using CRISPR-Cas9 mediated knock-outs, we are also investigating the function of polyp-specific JFT&rsquo;s and their role in prey-capture.</p>
<hr /><p><strong>Sea anemone Insulin-like peptides,</strong><span style="color: #202124;"><strong> For all time. Always.</strong></span></p>
<p><span style="color: #202124;">Alonso Delgado</span><span style="color: #202124;">1</span><span style="color: #202124;">, MaryMegan Daly</span><span style="color: #202124;">1</span></p>
<p><span style="color: #202124;">1</span><span style="color: #202124;">Ohio State University, Ohio, USA</span></p>
<p>Toxins are critical to the actiniarian biology, used for competition, prey capture and defense. Diversity in toxins can arise from duplications, mutations, and at times recruitment of nontoxic proteins. Insulin-like peptides (ILP) found in actiniarians show the potential in being recruited toxins. ILPS in actiniarians have poorly<span style="color: #222222;"> conserved insulin structures (Lu et al., 2020) and a conserved cysteine-rich pattern, a classic toxin pattern.&nbsp; ILP&rsquo;s with potential toxin function </span>have been documented across actiniarians: <em>Oulactis</em> sp (Mitchell et al 2021),<em> Nematostella vectensis</em> (Mitchell et al 2021)<em>, </em><span style="color: #222222;"><em>Exaiptasia pallida</em></span><span style="color: #222222;"> (Fu et al 2022), and in Clownfish hosting sea anemones (Delgado 2022 (in prep))</span></p>
<p><span style="color: #222222;">Due to the breadth of taxonomically distinct sea anemones where ILP was found, we hypothesize that ILP were recruited early in sea anemone phylogenetic history. We will also test selection signals across phylogenetic scales. In this study we reconstruct known and new sea anemone transcriptomes using Trinity, annotate protein families using PFAM, and conduct an independent blast search using known insulin-like toxins. We annotate all transcripts matching insulin-like proteins from our blast search using NCBI&rsquo;s NR database and Uniports/Swissport database (accessed 7/2022). Alignments and a phylogenetic tree and selection analysis of these toxins will then be used to infer their evolutionary history in sea anemones.</span></p>
<p><span style="color: #222222;">Fu, Jinxing, et al. "Transcriptome Sequencing of the Pale Anemones (Exaiptasia diaphana)&nbsp;</span></p>
<p><span style="color: #222222;">Revealed Functional Peptide Gene Resources of Sea Anemone." </span><span style="color: #222222;"><em>Frontiers in Marine Science</em></span><span style="color: #222222;"> 9 (2022): 856501.</span></p>
<p><span style="color: #222222;">Mitchell, Michela L., et al. "Identification, synthesis, conformation and activity of an insulin-like peptide from a sea anemone." </span><span style="color: #222222;"><em>Biomolecules</em></span><span style="color: #222222;"> 11.12 (2021): 1785.</span></p>
<p>Lu, A., Watkins, M., Li, Q., Robinson, S. D., Concepcion, G. P., Yandell, M., et al. (2020). Transcriptomic Profiling Reveals Extraordinary Diversity of Venom Peptides in Unexplored Predatory Gastropods of the Genus Clavus. Genome Biol. Evol. 12 (5), 684&ndash;700.</p>
<hr /><p><strong>The developmental potential of a single i-cell in </strong><strong><em>Hydractinia</em></strong></p>
<p><span style="text-decoration: underline;">&Aacute;ine Varley</span>, Gabriel Krasovec, Uri Frank</p>
<p>University of Galway, Ireland</p>
<p>The colonial hydroid <em>Hydractinia</em> has a migratory, proliferative population of stem cells known as i-cells. Past studies have shown that a population of i-cells can give rise to all somatic cell lineages as well as germ cells in <em>Hydractinia</em>. However, whether this population consists of distinct, lineage-committed i-cells or pluripotent cells has not yet been confirmed. My goal is to determine the developmental potential of a single i-cell in <em>Hydractinia.</em>&nbsp;&nbsp;&nbsp;&nbsp;</p>
<p>Making use of <em>Hydractinia</em>&rsquo;s remarkable growth, plasticity, I established a method for transferring a single i-cell from a double transgenic Piwi1::GFP Beta-Tubulin::mScarlet reporter colony into a wild type animal via stolonal contact. This method allowed tracing all progeny of a single grafted i-cell in the host tissue <em>in vivo</em>. We observed that wild type <em>Hydractinia</em> tissue possessing an initial single transgenic donor i-cell developed into a chimeric animal with a mixture of cells derived from the transgenic donor and the wild type recipient. Donor-derived cells, representing all major somatic lineages and germ cells were scattered in the recipient&rsquo;s tissues in both feeding and sexual polyps, and in the stolonal compartment. This study shows that <em>Hydractinia </em>possesses pluripotent i-cells, bringings a long-term debate to a final conclusion.</p>
<p>DuBuc, T. et al, (2020). Transcription factor AP2 controls cnidarian germ cell induction, Science.</p>
<p>Gahan, J. M. (2016). The interstitial stem cells in Hydractinia and their role in regeneration Current Opinion in Genetics and Development.</p>
<p>M&uuml;ller, W. A. (2004). Totipotent migratory stem cells in a hydroid, Developmental Biology.</p>
<p>Funding was provided by the National University of Ireland, Galway.</p>
<h3 style="text-align: center;">Invited Technology Speaker</h3>
<hr />

#### Session 3: Development I

<p><strong>The continuum of </strong><strong><em>Drosophila</em></strong><strong> embryonic development at single cell resolution</strong></p>
<p>Diego Calderon</p>
<p>University of Washington</p>
<p>Single cell technologies are a powerful new means to study metazoan development, enabling comprehensive surveys of cellular diversity at profiled timepoints, and shedding light on the dynamics of regulatory element activity and gene expression changes during the in vivo emergence of each cell type. However, nearly all such atlases of embryogenesis remain limited by sampling density, i.e. the number of discrete time points at which individual embryos are harvested. Given the rapidity with which molecular and cellular programs unfold, this limits the resolution at which regulatory transitions can be characterized. Dr. Calderon will report a continuous, single cell atlas of chromatin accessibility and gene expression that spans Drosophila embryogenesis, which in total includes nearly one million chromatin accessibility and half a million expression nuclei profiles from embryos collected in eleven tightly staged, overlapping windows. Leveraging the asynchronicity of embryos within each collection, Dr. Calderon and his colleagues developed a statistical model to estimate the age of each nucleus more precisely, resulting in continuous views of molecular and cellular transitions throughout embryonic development. Looking forward, this strategy may facilitate future investigations of in vivo gene regulation throughout embryogenesis at arbitrarily high temporal resolution in other model organisms.</p>
<hr /><p><strong>Population density modulates asexual reproduction &amp; gene expression in </strong><strong><em>Nematostella vectensis</em></strong></p>
<p><span style="text-decoration: underline;">Layla Al-Shaer</span><span style="text-decoration: underline;">1</span>, Noor Baban1, Michael Layden1</p>
<p>1Lehigh University, Bethlehem, Pennsylvania</p>
<p>Using the sea anemone <em>Nematostella vectensis</em>, we are uncovering the factors that promote asexual reproduction to further elucidate the mechanisms of cnidarian reproduction. First, the influence of size and food availability were considered. There was a positive correlation between adult size at time of cloning and the size of clones produced; adults reliably lost 30% in size to their clones. This suggests that <em>Nematostella </em>possess a set fission point which scales with size. Size had no effect on the time it took for cloning to first occur or the rate at which it occurred. Like other taxa, food restricted animals decreased in size and rarely produced clones, while well fed animals increased in size and often produced clones. Next, we asked if population density modulates <em>Nematostella</em> asexual reproduction by comparing the number of clones produced in high- and low-density mixed sex populations. Animals at low-density produced more clones than those from high-density populations. Increased byproducts of metabolism do not seem to explain this density-dependent difference in asexual reproduction. This suggests that <em>Nematostella</em> have a previously undescribed chemical recognition system that allows them to sense population density, and that they use this sensory information to alter their reproductive strategy. Finally, bulk RNA sequencing was done to gain insights about the molecular effect of density. Very few differentially expressed genes were found, suggesting that transcriptional regulation is not the primary mechanism controlling density-based differences in asexual reproduction. The few genes that were identified appear to be cnidarian specific and are expressed in a sex specific manner. The latter suggests that males and females have differential transcriptional responses to changes in density. Whether there are also sex specific behavioral differences in asexual reproduction based on density is being tested.</p>
<p>Funding: Lehigh University Faculty Innovation Grant; NIH RO1 GM127615.</p>
<hr /><p><strong>The transcription factor Zic4 prevents epithelial transdifferentiation in </strong><strong><em>Hydra</em></strong></p>
<p>Matthias Christian Vogg1, Jaroslav Ferenc2,3, Wanda Christa Buzgariu1, Chrystelle Perruchoud1, Panagiotis Papasaikas2,4, Paul Gerald Layague Sanchez1, Clara Nuninger2,3, Youn Le Cras2, C&eacute;line Delucinge-Vivier1, Leonardo Beccari5, St&eacute;phane Vincent6, Brigitte Galliot1,*, <span style="text-decoration: underline;">Charisios D. Tsiairis</span>2,*</p>
<p>1University of Geneva, Geneva, Switzerland.</p>
<p>2Friedrich Miescher Institute for Biomedical Research, Basel, Switzerland.</p>
<p>3University of Basel, Basel, Switzerland</p>
<p>4Swiss Institute of Bioinformatics, Basel, Switzerland</p>
<p>5University Claude Bernard Lyon, Lyon, France.</p>
<p>6Ecole Normale Sup&eacute;rieure de Lyon, Lyon, France.</p>
<p>The molecular mechanisms that maintain cell identities and prevent transdifferentiation remain mysterious. Regeneration offers the possibility for cells to dedifferentiate and acquire a different cell fate trajectory. Therefore, organisms that regenerate readily offer a fruitful paradigm to investigate the regulation of cell fate stability. Here, we used <em>Hydra</em> as a model system and show that Zic4 plays a key role in tentacle formation and tentacle maintenance. Its expression is dependent on Wnt/&beta;-catenin signaling, as well as the transcription factor Sp5, itself a downstream target of Wnt signaling. When <em>Zic4</em> expression is reduced, the epithelial cells of the tentacles, the battery cells, transdifferentiate into basal disc cells of the foot. In fact, the transformation of the tissue is such that the tentacle tips of the animals with reduced Zic4 can attach to solid substrates. The molecular cascade is also under investigation while the switch requires the re-entry into the cell cycle of tentacle battery cells and is accompanied by the degeneration of nematocytes embedded in these cells. Our data indicate that maintenance of cell fate by a Wnt-controlled mechanism is a key process during both homeostasis and regeneration.</p>
<hr /><p><strong>The developmental origin and relationship of the neuroglandular lineage in the sea anemone </strong><strong><em>Nematostella vectensis</em></strong></p>
<p>Ulrich Technau</p>
<p>Dept of Neurosciences and Developmental Biology, Research platform Single cell regulation of stem cells, University of Vienna, Djerassiplatz 1, 1030 Vienna</p>
<p>Interstitial cells are multipotent stem cells that give rise to neurons, nematocytes and gland cells and gametes in <em>Hydra</em>. However, they have so far only been identified in hydrozoans, despite the fact that other medusozoans and anthozoans show similar abilities to regenerate and to renew their tissues and cell types. In order to identify putative stem cell populations, we carried out single cell transcriptomics over many developmental stages of the sea anemone <em>Nematostella vectensis</em>. The investigation of scRNAseq libraries from several developmental stages enabled us to compare differentiation trajectories and molecular pathways between embryonic and adult stages. We identified several developmental regulators at key points of cell fate choices of neurons, gland cells and cnidocytes. We focused on a number of selected genes and generated transgenic lines to validate the predicted differentiation trajectories. We identify <em>soxC</em> as the most upstream regulator genes of the entire neuroglandular lineages. Knockdown of <em>soxC</em> abolishes the expression of neuronal, gland cell and nematocyte marker genes. Deeper analyses of the neurons identified numerous subpopulations, which are characterized by specific combinations of transcription factor and effector genes. We also aimed at identifying the putative stem cell population. Motivated by the expression pattern in single cell RNAseq, we generated transgenic lines of <em>piwi1 </em>and<em> nanos2, </em>which are also key marker genes of i-cells in <em>Hydra</em>. <em>Nanos2</em> and <em>Piwi1</em> reporter constructs show fluorophore expression numerous small cells located between epithelial cells and in all cells of the neuroglandular lineage, like <em>soxC</em>. In addition, the transgenes are strongly expressed in germline cells, as well as epithelial cells in the case of <em>Piwi1</em>. Around 23% of all cells are <em>Nanos2</em> positive. EdU stainings showed that at least a portion of these <em>nanos2</em> cells are cycling, but we also find EdU-positive cells that are negative for <em>nanos2</em>, indicating that <em>nanos2</em> cells are not the only dividing cell populations. To be able to ablate specifically the <em>nanos2</em> cells, we have established a nitroreductase line that we are currently investigating during regeneration. In summary, <em>nanos2</em> and <em>piwi1</em> is marking a cell population in <em>Nematostella</em> that might be homologous to the i-cells in <em>Hydra</em>.</p>

#### Session 4: Neurobiology
<p style="text-align: justify;"><strong>Characterizing neurogenesis pathways in </strong><strong><em>Hydra vulgaris</em></strong></p>
<p style="text-align: justify;"><span style="text-decoration: underline;">Abby S Primack</span>,1 Jack F Cazet,1 Jeff Farrell,2 Celina Juliano1</p>
<p style="text-align: justify;">1 Department of Molecular and Cellular Biology, UC Davis, Davis, CA, USA</p>
<p style="text-align: justify;">2 National Institute of Child and Human Development, NIH, MA, USA</p>
<p>In the small freshwater cnidarian <em>Hydra</em>, all differentiated cells in the homeostatic adult animal are replaced every three weeks, including the entire nervous system, using an active population of stem cells. These stem cells also enable <em>Hydra</em> to regenerate its nervous system following catastrophic injury. We ultimately aim to solve the gene regulatory networks that drive <em>Hydra</em> neurogenesis during both homeostasis and regeneration; in this presentation we will focus on homeostatic neurogenesis. Using single cell RNA-sequencing (scRNA-seq), we built a complete molecular map of the homeostatic <em>Hydra</em> nervous system; we have thus far sequenced ~30,000 single cell transcriptomes from neurons, neuronal progenitors, and interstitial stem cells. From these data, we identified 11 transcriptionally distinct neuron subtypes with unique molecular signatures and have determined the location of these subtypes in the <em>Hydra </em>nervous system using in situ hybridization and promoter-driven transgenic reporter lines. We have also built differentiation trajectories describing the neurogenesis of all 11 neuronal subtypes in an uninjured <em>Hydra</em>. This has allowed us to uncover the temporal expression of transcription factors (TFs) expressed during the differentiation of each neural subtype. Finally, we have collected ATAC-seq data to characterize the regulatory regions that are accessible in differentiated neurons. Together, these data are allowing us to build gene regulatory networks to describe neurogenesis in <em>Hydra</em>. As a next step, we will repeat this strategy for regenerative neurogenesis to find regulatory mechanisms specific to neural regeneration.</p>
<p><span style="color: #202124;"><strong>Investigating the role of </strong></span><span style="color: #202124;"><strong><em>NvashA</em></strong></span><span style="color: #202124;"><strong> during </strong></span><span style="color: #202124;"><strong><em>Nematostella </em></strong></span><span style="color: #202124;"><strong>neurogenesis</strong></span></p>
<p><span style="color: #202124;">Minghe Cheng</span><span style="color: #202124;">1</span><span style="color: #202124;">, Michael Layden</span><span style="color: #202124;">1</span></p>
<p><span style="color: #202124;">1</span><span style="color: #202124;">Lehigh university, Bethlehem, USA</span></p>
<p><span style="color: #202124;">Our lab is interested in comparing the mechanisms of developmental and regenerative neurogenesis in </span><span style="color: #202124;"><em>Nematostella. </em></span><span style="color: #202124;">Neurons are generated throughout development, but most studies only investigate patterning at gastrula stages.&nbsp; As a result, how neurons are patterned at later larval stages is poorly understood. Thus, here we focus on the </span><span style="color: #202124;"><em>achaete-scute</em></span><span style="color: #202124;"> homolog </span><span style="color: #202124;"><em>NvashA</em></span><span style="color: #202124;"> during neural development.&nbsp;</span></p>
<p><span style="color: #202124;">Despite a requirement for </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">to specify </span><span style="color: #202124;"><em>NvLwamide+</em></span><span style="color: #202124;"> neurons and </span><span style="color: #202124;"><em>Nvserum amyloid A+</em></span><span style="color: #202124;"> neurons in gastrulae, loss of </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">had no detectable impact on later born </span><span style="color: #202124;"><em>NvLWamide-like </em></span><span style="color: #202124;">or </span><span style="color: #202124;"><em>Nvserum amyloid A </em></span><span style="color: #202124;">expressing neurons, suggesting those neurons are regulated independently by </span><span style="color: #202124;"><em>NvashA</em></span><span style="color: #202124;"> in planulae. To gain a better insight about whether </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">still participates in neurogenesis at larval stages, we screened previously identified neuronal markers for changes in expression following loss of </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">in planulae. </span><span style="color: #202124;"><em>Nvcoup-like2 </em></span><span style="color: #202124;">is reduced, suggesting that </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">regulates these neurons in larval stages. Interestingly, </span><span style="color: #202124;"><em>Nvcoup-like2 </em></span><span style="color: #202124;">is not a target of </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">at the gastrula stage. These data suggest that </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">differentially regulates neuronal subtype marker expression during </span><span style="color: #202124;"><em>Nematostella </em></span><span style="color: #202124;">development. We are currently identifying </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">targets throughout development using bulk and single cell RNA sequencing approaches to identify genetic targets of and individual neuronal cell types regulated by </span><span style="color: #202124;"><em>NvashA</em></span><span style="color: #202124;">. Additionally, we are testing a dominant negative </span><span style="color: #202124;"><em>NvashA</em></span><span style="color: #202124;"> allele and optogenetic conditional expression strategies that will provide spatiotemporal control of </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">disruption that will allow us to better compare </span><span style="color: #202124;"><em>NvashA </em></span><span style="color: #202124;">function during neurogenesis and regeneration.&nbsp;</span></p>
<p><span style="color: #202124;">This project is funded by NSF CAREER 1942777 and NIH R01 GM127615.</span></p>
<hr /><p><strong>Studying neuronal mechanisms of somersaulting in </strong><strong><em>Hydra vulgaris</em></strong></p>
<p>Wataru Yamamoto and Rafael Yuste</p>
<p>Neurotechnology Center, Dept. of Biological Sciences, Columbia University, New York, New York</p>
<p><em>Hydra vulgaris</em>, a freshwater cnidarian, is a sedentary polyp that shows repeated contractions, as well as, somersaulting: an acrobatic locomotion performed by attaching the tentacles to the substrate and swinging the foot over the head to stand in a new position. How a distributed nervous system composed of a few hundred neurons can exercise the sensory-motor coordination to achieve such sophisticated behavior remains a mystery. This is partly because studying <em>Hydra</em> neuronal activity during somersaulting remains challenging, due to the difficulty of maintaining neurons in focus as they move in a large three-dimensional space. To address this, we used <em>Hydra</em> expressing the calcium indicator GCaMP6s to image neuronal activity during somersaulting performed in a confined space. By applying DeepLabCut-tracking methods based on deep convolutional networks, we found that the progression through most steps of somersaulting was not significantly different between free and confined preparations.</p>
<p>Using the confined preparation, we found that the activity of the rhythmic potential 1 (RP1) circuit, an ensemble of synchronously firing neurons distributed throughout the body, was significantly increased at the start of somersaulting. Activation of basal disc muscles and nematocytes on the tentacles also occurred at this time point. To investigate the role of RP1 activity in somersaulting, we altered RP1 activity. Firstly, reducing the total number of neurons with 0.04% colchicine abolished somersaulting. Secondly, increasing the osmolarity of the medium decreased RP1 activity and generated a 63.6 % decrease in the number of somersaults. Our results indicate that RP1 neurons control basal disc muscles and nematocytes in precise order and timing, leading to somersaulting.&nbsp;</p>
<p><em>Supported by NSF (CRCNS 1822550; 2203119) and Vannevar Bush Faculty Award (ONR N000142012828).</em></p>
<hr /><p><strong>Phototaxis is a state-dependent complex behavior in </strong><strong><em>Hydra vulgaris</em></strong></p>
<p><span style="text-decoration: underline;">Soonyoung (Kelly) Kim</span><span style="text-decoration: underline;">1</span>, Jacob T. Robinson1,2,3</p>
<p>1 Department of Electrical and Computer Engineering, Rice University, Houston, Texas, USA</p>
<p>2 Department of Bioengineering, Rice University, Houston, Texas, USA</p>
<p>3 Department of Neuroscience, Baylor College of Medicine, Houston, Texas, USA</p>
<p>Understanding how internal states like satiety relate to animal behavior is a fundamental question in neuroscience. <em>Hydra vulgaris</em>, a freshwater cnidarian with only eleven neuronal cell types, provides a tractable model system for studying state dependent behaviors. We find that starved <em>Hydra</em> display robust phototaxis, while the phototactic behavior in fed <em>Hydra</em> is less consistent. We then model this behavior as a set of 3 simple sequences, and ask which of these are affected by satiety. We find that satiety only alters one of these processes, namely, the probability that the animal jumps to a new position. These findings yield insights into how a simple organism, <em>Hydra</em>, encodes internal states that manifest in behavioral changes, as well as general principles for studying the relationship between state-dependent behaviors and their underlying molecular mechanisms.</p>

#### Session 5: Evo Devo I
<hr /><p><strong>Branching morphogenesis of </strong><strong><em>Cladonema</em></strong><strong> medusa tentacles</strong></p>
<p>Shiting Hou1, Akiyo Fujiki1, Ayaki Nakamoto2, and <span style="text-decoration: underline;">Gaku Kumano</span>1</p>
<p>1Asamushi Research Center for Marine Biology, Tohoku University, Aomori, Japan</p>
<p>2Aomori University, Aomori, Japan</p>
<p>The interaction between the epithelial cell layer and its underlying mesoderm-derived tissues plays a critical role in shaping the tissues/organs in many triploblastic animals. Inductive signals, such as receptor tyrosine kinase (RTK) signaling, act on cells of the epithelium to initiate 3-dimensional changes. However, how tissues are shaped in a diploblastic animal with no mesoderm remains largely unknown. In this study, the jellyfish <em>Cladonema pacificum </em>was used to investigate branch formation. Its medusa tentacles undergo branching, which increases the epithelial surface area carrying nematocytes, thereby maximizing prey capture. It was found that new branches were successively created one after another at the proximal region of the main tentacle while the main tentacle grows. At the new branching sites, hydrozoan-specific pluripotent stem cells, namely interstitial cells (I-cells), were periodically accumulated. During branch elongation, the accumulated I-cells remained located at the tip of the growing branches, while proliferating and leaving behind their differentiating descendant cells. Finally, fibroblast growth factor (FGF) signaling was found to regulate branch elongation. This study highlights an essential role for I-cells in the tissue-shaping morphogenesis of a diploblastic animal. In addition, it identifies patterning through repeated applications of a single rule, a mechanism involving RTK signaling and cell proliferative activity at the branch tip for branching morphogenesis apparently conserved across the animal kingdom.</p>
<p>Fujiki, A., Hou, S., Nakamoto, A., and Kumano, G. (2019). Branching pattern and morphogenesis of medusa tentacles in the jellyfish <em>Cladonema pacificum</em> (Hydrozoa, Cnidaria). Zoological Lett. 5: 12.</p>
<p>Hou, S., Zhu, J., Shibata, S., Nakamoto, A., and Kumano, G. (2021). Repetitive accumulation of interstitial cells generates the branched structure of <em>Cladonema</em> medusa tentacles. Development 148: dev199544.</p>
<hr /><p><strong>Inducible organs of aggression in the plumose sea anemone, </strong><strong><em>Metridium senile, </em></strong><strong>as a model for the evolution of cell type plasticity</strong></p>
<p><span style="text-decoration: underline;">Yareli Alvarez</span>1 and Leslie S. Babonis1</p>
<p>1Cornell University, Ithaca, New York, USA</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;Sea anemones can reproduce by asexual fission creating colonies of genetically identical individuals. Some species of sea anemones, such as <em>Metridium senile</em>, respond aggressively to members of different colonies by transforming preexisting feeding tentacles into specialized fighting tentacles that possess a unique collection of stinging cells. Because of this unique suite of cnidocytes, fights end in necrosis and either death or retreat of the encroaching individual. Inducible defensive structures have evolved independently in cnidarians multiple times, suggesting plastic control of stinging cell identity may be an ancestral feature of this group. To understand how cell fate plasticity is controlled, we induced fighting responses in pairs of non-identical <em>M. senile</em> adults and collected samples of feeding tentacles and fighting tentacles at multiple time points after induction. We use microscopy to characterize the morphology of stinging cells in tentacles transitioning from feeding to defensive structures and comparative transcriptomics to identify the regulatory pathways driving stinging cell plasticity. Together, our results elucidate the mechanism by which environmental cues can be transduced into novel traits and serve as a model for understanding cell fate plasticity more broadly.</p>
<p>This research was funded by Friday Harbor Laboratories, University of Washington and the department of Ecology and Evolutionary Biology, Cornell University.</p>
<hr /><p><strong>Cnidarian-specific neuropeptide RPamide decelerates the timing of life cycle transition in the sea anemone </strong><strong><em>Nematostella vectensis</em></strong></p>
<p>Julia Baranyk and Nagayasu Nakanishi</p>
<p>University of Arkansas, Fayetteville</p>
<p>Neuropeptides constitute a class of small signalling molecules responsible for regulating physiology and behaviour across the animal kingdom. One of the neuropeptides unique to the phylum Cnidaria is RPamide. In the sea anemone <em>Nematostella vectensis,</em> the RPamidergic sensory nerve net develops in the aboral ectoderm of the planula larva. During planula development, RPamide becomes expressed in pharyngeal ectodermal sensory cells, as well as in a subset of endodermal neurons. At life cycle transition, the RPamidergic sensory nerve net of aboral ectoderm degenerates by apoptosis and an RPamidergic sensory nerve net develops in the oral ectoderm of a polyp. These expression data are consistent with the role of RPamide in regulating larval behavior, metamorphosis, and/or polyp behavior. Here, we investigate the functional role of RPamide indevelopment, metamorphosis, and growth of <em>N. vectensis.</em> Using CRISPR-Cas9-mediated mutagenesis, we generated knockout mutant lines for the RPamide gene. We confirmed by immunohistochemistry with an anti-RPamide antibody that RPamide immunoreactivity was absent in RPamide knockout mutants. Analyses of developmental timing show that RPamide knockout mutants undergo more rapid progression through metamorphosis and accelerated growth relative to their heterozygous or wildtype siblings. Our results suggest that RPamide plays a role in negatively regulating initiation and progression of development and metamorphosis in <em>N. vectensis.</em></p>
<p>Zang H, Nakanishi N. (2020). Expression analysis of cnidarian-specific neuropeptides in a sea anemone unveils an apical-organ-associated nerve net that disintegrates at metamorphosis. <em>Front Endocrinol </em>11:63.</p>
<hr /><p><strong>A cell-type atlas from a scyphozoan jellyfish </strong><strong><em>Aurelia</em></strong><strong> sp. provides insights into cell-type diversity and reveals conserved cell specification mechanisms across the Cnidaria</strong></p>
<p><span style="text-decoration: underline;">Alison G. Cole</span>, Oliver Link, Juan Daniel Montenegro, Ulrich Technau</p>
<p>University of Vienna, Dept. Neurosciences and Developmental Biology; Djerassiplatz 1, 1030 Vienna, AT</p>
<p>One challenge in evolutionary science is understanding cell-type diversification processes as a proxy for estimating the emergence of cell-type complexity. Basally branching animal phyla, such as cnidarians, sponges and ctenophores, are uniquely positioned to contribute to this understanding. Here, we present the first cell-type catalogue across the entire life-history from a cosmopolitan scyphozoan, the moon-jellyfish (<em>Aurelia sp.</em>). We generated single-cell RNAseq libraries across all life history stages and demonstrate an increase of cellular complexity during the transition from the sessile polyp to the free-swimming medusa. We identify cell transcriptomes of the neuro-muscular unit of the medusa that are absent in the sessile polyp, marking the emergence of new cell types in the medusa. A prominent medusazoan cell type is the striated muscle profile, which includes a number of myosin-heavy chain orthologs not expressed in the smooth muscle. Comparisons with an atlas of similar composition from the sea anemone <em>Nematostella vectensis</em> reveals common specification pathways, including but not limited to the cnidocyte lineages. Both species demonstrate divergence of cnidocyte transcriptomic profiles that later converge to common mature profile. Similar sets of transcription factors are involved in this transition between species. Our datasets provide a foundation for further research on the evolution of cell types.</p>

### Friday, September 9

#### Session 6: Physiology I
<p><strong>Early-hour molecular responses of wound healing in </strong><strong><em>Pocillopora damicornis</em></strong></p>
<p><span style="text-decoration: underline;">Allyson DeMerlis</span>1,2, Samantha Beasley1, Kevin Rodriguez3, Nikki Traylor-Knowles1</p>
<p>1Department of Marine Biology and Ecology, Rosenstiel School of Marine and Atmospheric Science, University of Miami, Miami, FL, USA</p>
<p>2Atlantic Oceanographic and Meteorological Laboratory, National Oceanic and Atmospheric Administration, Miami, FL, USA</p>
<p>3Department of Ecology and Evolutionary Biology, University of California Los Angeles, Los Angeles, CA, USA</p>
<p>Reef-building corals demonstrate a high capacity for tissue growth and regeneration, an integral part of their life history strategy as sessile colonial organisms. Tissue regeneration is necessary in response to injury, which may be caused by competition or predation, and a coral&rsquo;s ability to rapidly heal its wounds dictates its chance at survival. Previous work in cnidarians has identified the molecular responses which orchestrate tissue regeneration; however, the early stages of this response are not well-understood, as the sampling methodologies described are predominantly on the order of days post-injury. We hypothesize that the first hours of wound healing involve the activation of the coral innate immune system to prevent infection at the injury site. To address this, we wounded 18 fragments of one genotype of <em>Pocillopora damicornis</em> and then collected tissue samples at each hour, up to four hours post-injury, for coral host differential gene expression analysis in comparison to unwounded controls. Preliminary results identified distinct gene cluster profiles over the four-hour time series for wounded versus controls, including genes implicated in the innate immune response. Overall, our findings provide new insights into the molecular mechanisms implicated in the early hours of scleractinian wound healing.</p>
<hr /><p><strong>Algal symbiont presence and not photosynthetic contribution affects the development of </strong><strong><em>Exaiptasia diaphana</em></strong><strong> asexual offspring</strong></p>
<p>Samuel A. Bedgood1, Keyla Plichon1,2, Virginia M. Weis1</p>
<p>1Oregon State University, Corvallis, Oregon USA</p>
<p>2Universit&eacute; C&ocirc;te d'Azur, Nice, France</p>
<p>We followed the development of asexual offspring produced by pedal laceration in the sea anemone <em>Exaiptasia diaphana</em>, commonly called Aiptasia. Recent work by Presnell et al. (2022) has shown that offspring with algal symbionts (symbiotic) develop faster than those without (aposymbiotic), having more tentacles after three weeks of development. It was proposed that nutritional contributions from the algae could provide the resources needed to develop faster. We tested this hypothesis by inoculating pedal lacerates with native, non-native, and native but heterotrophic mutant symbionts (provided by the Jinkerson Lab, Jinkerson et al. 2022). We chose these symbiont types because they provide high, low, and no photosynthetic contributions to the host, respectively. We hypothesized that development rate would depend on the contribution of symbionts to the host where those anemones with heterotrophic mutant symbionts would have the same development rate as aposymbiotic anemones. Contrary to our hypothesis, we found that symbiont identity had no effect on development; all symbiotic anemones developed faster than aposymbiotic anemones in the first two weeks, regardless of symbiont contribution or density. Our work suggests host-symbiont signaling and not nutrition drives the difference in development rate between symbiotic and aposymbiotic offspring.</p>
<p>Jinkerson RE, Russo JA, Newkirk CR, Kirk AL, Chi RJ, Martindale MQ, Grossman AR, Hatta M, Xiang T (2022) Cnidarian-Symbiodiniaceae symbiosis establishment is independent of photosynthesis. Curr Biol 32:2402-2415.e4. doi: 10.1016/j.cub.2022.04.021</p>
<p>Presnell JS, Wirsching E, Weis VM (2022) Tentacle patterning during <em>Exaiptasia diaphana</em> pedal lacerate development differs between symbiotic and aposymbiotic animals. PeerJ 10:e12770. doi: 10.7717/peerj.12770</p>
<p>We thank NSF for funding this project (grant number 2124119 to V.M.W.).</p>
<hr /><p><strong>How does </strong><strong><em>Nematostella</em></strong><strong>&rsquo;s circadian clock behave under conflicting light and temperature cycles?</strong></p>
<p><span style="text-decoration: underline;">Cory Berger</span>1,2, Ann Tarrant1</p>
<p>1Biology Department, Woods Hole Oceanographic Institution, Woods Hole, MA, United States</p>
<p>2MIT-WHOI Joint Program in Oceanography/Applied Ocean Science &amp; Engineering, Cambridge and Woods Hole, MA, USA</p>
<p>Like most organisms, cnidarians use circadian clocks to infer the time of day from cyclic environmental factors such as light and temperature. These factors, called zeitgebers, are usually studied in isolation, although organisms face the task of integrating information from multiple co-occurring zeitgeber cycles in nature. How do clocks behave when misaligned zeitgebers provide conflicting information (&ldquo;sensory conflict&rdquo;)? Such a situation can disrupt circadian rhythms, limiting the range of conditions under which normal behavior can occur. Alternatively, clocks may prioritize information from one zeitgeber over the other. There are currently few comprehensive multi-zeitgeber experiments in any animal, and we do not understand how sensory conflict affects rhythmic gene expression. To address these questions, we integrate behavioral experiments and RNA sequencing in <em>Nematostella vectensis</em>, a cnidarian model for circadian biology. Using a comprehensive range of light and temperature cycles, we show thatcircadian behavior is disrupted under extreme sensory conflict, in a fashion that depends on the specific phase relationship between zeitgebers. Sensory conflict substantially altered the rhythmic transcriptome and disrupted the expression of metabolic gene modules. However, hundreds of genes also gained rhythmic expression and core clock genes remained rhythmic, suggesting a surprising robustness of transcriptional rhythmicity. Temperature cycles seem to be the dominant driver of rhythmic gene expression in this animal, although light exerts strong direct effects on behavior.</p>
<p><span style="color: #222222;">Berger, C. A., &amp; Tarrant, A. M. (2022). Sensory conflict disrupts circadian rhythms in the sea anemone </span><span style="color: #222222;"><em>Nematostella vectensis</em></span><span style="color: #222222;">. </span><span style="color: #222222;"><em>bioRxiv</em></span><span style="color: #222222;">, https://doi.org/10.1101/2022.04.11.487933</span></p>
<p>This work was supported by WHOI&rsquo;s Ocean Ventures Fund.</p>
<hr /><p><strong>Mechanical characterization of regenerating </strong><strong><em>Hydra vulgaris</em></strong><strong> tissue pieces</strong></p>
<p>Thomas Perros1, Ana&iuml;s Reboullet1, Ana&iuml;s Biquet-Bisquert1, Morgan Delarue2, Pierre Joseph2, Philippe Marcq3, <span style="text-decoration: underline;">Olivier Cochet-Escartin</span><span style="text-decoration: underline;">1</span></p>
<p>1Institute for Light and Matter, CNRS, University Lyon1, Lyon, France.</p>
<p>2Laboratory for Analysis and Architecture of Systems, CNRS, Toulouse, France.</p>
<p>3Physics and Mechanics of heterogeneous media, CNRS, Sorbonne University, ESPCI, Paris, France.</p>
<p>The spontaneous axial patterning of regenerating <em>Hydra</em> tissue pieces or aggregates has long been an open question at the interface of developmental biology and biophysics. Historically, this question has been tackled through biochemical pattern formation and reaction-diffusion processes. In recent years though, it has become increasingly clear that tissue mechanics and cellular stretching also play a role in defining new organizers and axes. Our current representation of this regenerative process is thus a mechano-biological one. However, the mechanical properties and behavior of these samples have never been measured, limiting our capacity to decipher and quantify mechano-biological couplings. Using an original microfluidic approach allowing for multiplexed micro-aspiration experiments, we carried a rheological characterization of regenerating <em>Hydra</em> tissue pieces. We found that they behave as yield stress fluids: deforming as elastic shells at low stresses and as visco-elastic fluids at large ones and we quantified all their relevant properties. These results open the door to a quantitative study and modelling of feedback loops between mechanics and biochemistry during patterning. Finally, our microfluidic device also allows to apply external forces on regenerating samples on the time scale of days. Doing so, we observe defects in patterning and study the organism&rsquo;s response to these mechanical cues at the molecular level.</p>
<p>Funding: MITI at CNRS, iLM, FRAMA.</p>
<p>Ferenc, J. <em>et al.</em> Mechanical oscillations orchestrate axial patterning through Wnt activation in Hydra. <em>Sci. Adv.</em><strong>7</strong>, 6897 (2021).</p>
<h3>Invited Technology Speaker:&nbsp;</h3>

#### Session 7: Physiology II

<hr /><p><strong>New approaches to structural biology using single particle cryoEM</strong></p>
<p>James Letts</p>
<p>University of California, Davis</p>
<p>Dr. Letts is interested in understanding the mechanism of electron transport membrane proteins, a varied class of enzymes essential for energy transduction and cellular defense/signaling. Research in his lab spans both the basic biology of these enzymes employing multiple model organisms and translational applications to human health and disease. The major methods employed by the Letts Lab involve developing strategies for the extraction and stabilization of large membrane protein complexes followed by their functional and structural characterization. Specifically the Letts Lab is using single particle cryoEM to push the boundaries of what can be achieved from limited (low abundance) samples and pioneering a &ldquo;bottom-up&rdquo; structural proteomics approach to explore the diversity of eukaryotic mitochondrial bioenergetics. These approaches have led to the first structures of mitochondrial respiratory complexes from plants and ciliates which revealed both universally conserved and divergent aspects of this core metabolic process.</p>
<hr /><p><strong>Nutritional regulation of growth in the sea anemone </strong><strong><em>Nematostella vectensis</em></strong></p>
<p><span style="text-decoration: underline;">Kathrin Garschall</span><span style="text-decoration: underline;">1</span>, Ian Kouzel*2 Eudald Pascual-Carreras*1, Ragnhild Valen1, Daria Filimonova1, and Patrick R.H. Steinmetz1</p>
<p>1 Sars International Centre for Marine Molecular Biology, Bergen, Norway</p>
<p>2 Systems Biology &amp; Biomedical Data Science Laboratory, University of Konstanz, Germany</p>
<p>*Equal contributions</p>
<p>Most marine invertebrates, including sea anemones, show feeding-dependent growth throughout their life. The major genetic models, however, have decoupled feeding and growth to keep a fixed adult body size. To gain insight into the nutritional regulation of growth, we studied how the sea anemone <em>Nematostella vectensis</em> responds to feeding and starvation on the organismal, transcriptomic and cellular level.</p>
<p>Juvenile <em>N. vectensis</em> show feeding-dependent growth and dramatic starvation-induced shrinkage (&plusmn;90% within 20 weeks). RNAseq analysis uncovered a gradual transition between growth and shrinkage processes: within two days after feeding, the transcriptome is characterized by genes for transcription, translation, and cell division. Beyond five days of starvation, we saw increased expression of pro-apoptotic and tumor necrosis factor signaling genes. Supporting our findings by flow cytometry and confocal microscopy, we found an increase in S-phase cells and a corresponding decrease in G1-phase cells within 24h post feeding. Only 24h later, the level of S-phase cells drops 3-fold and remains low. Re-feeding leads to a rapid increase in S-phase cells, suggesting that a significant population of proliferative cells adopts a quiescent state upon starvation and awaits nutritional input to re-enter the cell cycle. During shrinkage, we observed a decrease of cell numbers, median cell size and increased cell death, which together likely contribute to a reduction in body size.</p>
<p>We demonstrated in <em>Nematostella</em> that feeding-dependent organismal growth and shrinkage are the outcomes of a tight nutritional regulation of the cell cycle, cell death and the underlying genes. Our findings thus provide exciting opportunities for investigating the mechanisms and evolution of life-long, feeding-dependent growth.</p>
<hr /><p><strong>Saved by the Cell: Immune cells and Stony Coral Tissue Loss Disease (SCTLD)</strong></p>
<p><span style="text-decoration: underline;">Lys M. Isma</span><span style="text-decoration: underline;">1</span>, Grace A. Snyder1, Nikki Traylor-Knowles1</p>
<p>1University of Miami Rosenstiel School of Marine and Atmospheric Sciences</p>
<p>In 2014 a new coral disease was discovered termed Stony Coral Tissue Loss Disease (SCTLD) within and near Port Miami. The cause of SCTLD is unknown, but there is evidence that bacteria or viruses associated with the Symbiodiniaceae may be implicated. SCTLD affects over 20 species of Caribbean coral, however, not all coral species have the same susceptibility. This could be due to the diversity in coral immune systems. In this study, I will use fluorescence activated cell sorting (FACS) to investigate the ratio of immune cells to the total cell population in six healthy coral species and examine how it correlates to known SCTLD susceptibility. I hypothesize that susceptibility is linked to immune cell populations and that highly susceptible species have lower ratios of immune cells compared to their total cell populations. I will examine immune cell ratios of two known highly susceptible species <em>P.strigosa</em> and <em>P.clivosa</em>, two intermediately susceptible species <em>O.faveolata</em><em>and M.cavernosa</em> and lowly susceptible species <em>P.astreoides</em> and <em>A.cervicornis</em>. We will compare this to what is known about coral SCTLD susceptibility and examine the functionality of the immune cells associated with these coral species. From this work we will have a better grasp of SCTLD susceptibility and the role of the immune system in this pathology.&nbsp;</p>
<p>Funding</p>
<p>National Science Foundation</p>
<p><span style="color: #202124;">United States &ndash; Israel Binational Science Foundation</span></p>
<p>References</p>
<p>Aeby, G. S., Ushijima, B., Campbell, J. E., Jones, S., Williams, G. J., Meyer, J. L., H&auml;se, C. &amp; Paul, V. J. (2019). <em>Front Mar Sci</em>. <strong>6</strong>,.</p>
<p>Palacio-Castro, A. M., Dennison, C. E., Rosales, S. M. &amp; Baker, A. C. (2021). <em>Coral Reefs</em>. <strong>40</strong>, 1601&ndash;1613.</p>
<p>Palmer, C. v. &amp; Traylor-Knowles, N. (2012). <em>Proceedings of the Royal Society B: Biological Sciences</em>. <strong>279</strong>, 4106&ndash;4114.</p>
<p>Snyder, G.A., Eliachar, S., Connelly, M. T., Talice, S., Hadad, U., Gershoni-Yahalom, O., Browne, W.E.., Palmer, C. v., Rosental, B. &amp; Traylor-Knowles, N. (2021). <em>Frontiers in Immunology</em>. <strong>12</strong>,.</p>
<hr /><p><strong>The cnidarian antiviral immune system reflects ancestral complexity</strong></p>
<p><span style="text-decoration: underline;">Yehu Moran</span><span style="text-decoration: underline;">1</span></p>
<p>1The Hebrew University of Jerusalem, Israel</p>
<p>The extremely fast co-evolution of viruses and host antiviral systems can result in blurry homology or even in shifts of whole defense mechanisms between host species. In vertebrates, the antiviral immunity is heavily based on the interferon pathway whereas in the case of invertebrates the antiviral immunity is believed to be based mostly on an RNA interference (RNAi). Until now, the recognition mechanism and mode of action of such systems were studied mostly in vertebrates, insects and nematodes. From this limited phyletic sampling, it is impossible to deduce what was the original mode of action of these systems in their last common ancestor and how antiviral immunity was triggered in early animals. To attain novel insights into the evolution of this system, we study it in an outgroup: the sea anemone <em>Nematostella vectensis</em>. We harness the genetic and molecular tools available for this species to decipher the cnidarian system for battling RNA viruses and answer the outstanding questions regarding the evolution of antiviral immunity and its ancestral state in animals. We show that like bilaterian animals <em>Nematostella</em> reacts transcriptionally and proteomically to the viral hallmark of long (200-7000 bp) double-stranded RNA (dsRNA). However, unlike vertebrates and nematodes, <em>Nematostella</em> is not differentially-responsive to short and long dsRNA carrying or lacking the viral hallmark of 5&prime;-triphosphate group. Our results for long dsRNA put in question the textbook dichotomy between the antiviral immune systems of vertebrates and ecdysozoans as we find upregulated components of both systems in <em>Nematostella</em>. These findings support the intriguing scenario that the ancient intracellular antiviral innate immunity system which was present in the last common ancestor of Cnidaria and Bilateria was in several aspects more complex and diverse than the systems found in extant vertebrates and protostomes such as arthropods and nematodes.</p>
<p>We acknowledge the support of European Research Council consolidator grant 863809</p>

#### Session 8: Cell Biology I

<p style="text-align: justify;"><strong>Visualizing the expression of Alr1 in </strong><strong><em>Hydractinia symbiolongicarpus</em></strong></p>
<p style="text-align: justify;"><span style="text-decoration: underline;">Manuel H. Michaca</span>1, Steven M. Sanders1, Srija Konduru1, Matthew L. Nicotra1</p>
<p style="text-align: justify;">1Starzl Transplantation Institute, University of Pittsburgh, Pittsburgh, PA, USA</p>
<p>Allorecognition is the ability of an organism to recognize self from non-self within the same species. Allorecognition is nearly ubiquitous in colonial marine invertebrates, where it regulates intraspecific spatial competition and prevents stem cell parasitism. Our lab aims to expand our knowledge of invertebrate allorecognition by studying it in <em>Hydractinia symbiolongicarpus</em>. Our work has revealed that Allorecognition (<em>Alr</em>) 1 and <em>Alr2</em> genes are determinants of responses in <em>Hydractinia</em>, where colonies matching at least one allele at both genes can fuse while complete mismatches lead to rejection. Matching Alr1 and Alr2 alleles homophilically bind across cells through their extracellular Ig domains. However, the intracellular events that take place after this binding are as yet unknown. To investigate this, a yeast two-hybrid screen was performed and three proteins that interact with Alr1&rsquo;s cytoplasmic tail were identified: <em>Afadin</em>, <em>PICK1</em>, and <em>Syntenin</em>. These PDZ proteins are present in vertebrates and regulate cell-cell adhesion, suggesting Alr1 may play a similar role in <em>Hydractinia.</em> Here we report the development of Alr1 antibodies and their use in immunolocalization experiments across different life stages of <em>Hydractinia. </em>Our results show that Alr1 localizes to cell membranes in all tissues, consistent with a constitutive role in cell-cell adhesion. This suggests part of the <em>Hydractinia </em>allorecognition system may have evolved from existing cell adhesion proteins.&nbsp;</p>
<hr /><p><strong>Parasitic strategies in Cnidaria from omics perspective. Case study myxozoan parasite Sphaerspora molnari.</strong></p>
<p><span style="text-decoration: underline;">Anush Kosakyan</span><span style="text-decoration: underline;">1</span>, Monika <span style="color: #202124;">Wiśniewska</span><span style="color: #202124;">1</span>, Gema-Alama Bermejo<span style="color: #202124;">1</span>, Anna Nenarokova1, Ron Dirks2 and Astrid S. Holzer1,3</p>
<p>1Institute of Parasitology, Biology Center of the Czech Academy of Sciences, Czech Republic</p>
<p>2Future Genomics Technologies, the Netherlands</p>
<p>3 University of Veterinary Medicine, Austria</p>
<p>Myxozoans are metazoan parasites that evolved from their free-living cnidarian ancestors to obligate microscopic endoparasites around 600 mya. In addition to their fascinating evolutionary history, they are of interest to many scientists because some cause serious diseases in farmed fish and wild fish populations.</p>
<p><em>Sphaerospora molnari</em> is a myxozoan parasite that infects common carp and causes mortality in central European pond cultures. We sequenced and analyzed genomic and transcriptomic data from <em>S. molnari</em> and combined our results with existing omics datasets from cnidarians to understand which functional gene groups are associated with successful parasitic strategies in S. molnari and in myxozoans in general.</p>
<p>We combined two different technologies: PromethION-Oxford Nanopore and Illumina, to be able to analyze the highly derived myxozoan genome and its structure. We also used a combination of commonly used algorithms, custom-made scripts and pipelines for assembly, structural and functional annotation of the genome. We performed comparative genome analysis using datasets from free-living cnidarians and myxozoans to describe Myxozoan lineage specific genes.</p>
<p>Our results showed that most myxozoan lineage-specific genes are related to parasitic strategies, and many of them are duplicated in the <em>S. molnari</em> genome. We identified homologs of many of these genes in other parasitic organisms (e.g., <em>Plasmodium</em>, <em>Giardia</em>, <em>Trypanosoma</em>) that are essential for their pathogenicity and successful survival in their hosts and proposed a list of "pathogenicity-related" gene families.</p>
<p style="text-align: justify;">Funding:&nbsp; Czech Science Foundation (#20-30321Y, 19-28399X)</p>
<hr /><p><strong>Extracellular matrix dynamics during the life history of Nematostella vectensis</strong></p>
<p><span style="text-decoration: underline;">Bruno Gideon Bergheim</span><span style="text-decoration: underline;">1</span> and Suat &Ouml;zbek1</p>
<p>1Center for Organismal Studies, Ruprecht-Karls-Universit&auml;t Heidelberg, Heidelberg, Germany</p>
<p>While cellular dynamics during development have received much attention over the years, the influence of the extracellular matrix (ECM) on developmental processes remains largely unknown. Cnidarians are the earliest animals that possess all core components of the animal ECM and their simple body plan combined with a complex, multi-stage life cycle makes them ideal model organisms to study the role of the ECM on morphogenesis. We present the first proteomic map of the ECM of the starlet sea anemone <em>Nematostella vectensis</em> throughout multiple stages of its complex life cycle. Through quantitative proteomic studies of the isolated ECM from larvae, primary polyp and adult we identified an unexpectedly rich set of proteins that constitute the matrisome during each stage of development. We also introduce a novel set of antibodies to visualize major ECM components like Collagen type I, IV and Laminin. Finally, we performed elasticity measurements of isolated mesoglea using atomic force microscopy to define the biophysical adaptations of the ECM to the different life stages.</p>
<p><span style="color: #303030;"><strong>A </strong></span><span style="color: #303030;"><strong><em>Hydra</em></strong></span><span style="color: #303030;"><strong> Claudin cell-cell contact protein is involved in epithelial tissue dynamics, regeneration, and osmoregulation</strong></span></p>
<p><span style="color: #303030;">Marie-Kristin Eder</span><span style="color: #303030;">1</span><span style="color: #303030;">, Matthias Achrainer</span><span style="color: #303030;">1</span><span style="color: #303030;">, Kevin Gr&uuml;ner</span><span style="color: #303030;">1</span><span style="color: #303030;">, Adolf Sandbichler</span><span style="color: #303030;">1</span><span style="color: #303030;">, Willi Salvenmoser</span><span style="color: #303030;">1</span><span style="color: #303030;">, Jack Cazet</span><span style="color: #303030;">2</span><span style="color: #303030;">, Celina Juliano</span><span style="color: #303030;">2</span><span style="color: #303030;">, and </span><span style="color: #303030; text-decoration: underline;">Bert Hobmayer</span><span style="color: #303030;">1</span></p>
<p><span style="color: #303030;">1</span><span style="color: #303030;">Institute of Zoology, University of Innsbruck, Innsbruck, Austria</span></p>
<p><span style="color: #303030;">2</span><span style="color: #303030;">University of California, Davis, USA</span></p>
<p>Claudins are major components of apical junctional complexes in bilaterians. Here, we identified 33 orthologues of the <em>claudin</em> gene family in <em>Hydra</em>. Single-cell transcriptome expression patterns suggest that this eumetazoan expansion of the <em>claudin</em> gene family correlates with the formation of cell-cell contacts between diversified numbers of cell types. In order to determine the principal functions of Claudin proteins in <em>Hydra</em>, we carried out a detailed analysis of protein localization of <em>Hydra </em>Claudin1, which is most strongly expressed in the ectodermal stem cell lineage. Transgenic <em>Hydra</em> expressing a Claudin1-GFP fusion protein demonstrated specific protein localization in apical septate junctions throughout the entire polyp body. To test the function of Claudin1, we used an optimized siRNA protocol to efficiently knock-down <em>claudin1 </em>in the ectodermal tissue layer. <em>claudin1</em> knock-down caused disruption of epithelial organization and a disturbed ultra-structure of septate junctions. In addition, the ability of the ectodermal epithelial layer to maintain normal osmoregulation was reduced. Dynamic 3D Claudin1 structure simulations predict formation of septate junction-specific pores for selective transport. Furthermore, knock-down animals exhibit strongly reduced capacities for head and foot regeneration, and detailed expression analysis suggests that Claudin1 acts in head- and foot-specific terminal differentiation. In conclusion, our study is the first detailed structural and functional characterization of a pre-bilaterian Claudin cell-cell-contact protein acting in epithelial stem cells to maintain tissue integrity and allowing for normal morphogenesis.</p>
<p><span style="color: #303030;">This research was supported by the EC H2020 Marie Curie COFUND research grant No 847681.</span></p>

#### Session 9: Cell Biology II

<p><span style="color: #303030;"><strong>Evolutionary dynamics of the stem cell factor Myc and functional conservation of the unique </strong></span><span style="color: #303030;"><strong><em>Hydra</em></strong></span><span style="color: #303030;"><strong> Myc3 protein</strong></span></p>
<p><span style="color: #303030; text-decoration: underline;">Marion Lechable</span><span style="color: #303030;">, Xuechen Tang,</span><span style="color: #303030;">Moses Kibet, Bernhard Egger, Monica Fern&aacute;ndez-Quintero, Markus Hartl and Bert Hobmayer.</span></p>
<p><span style="color: #303030;">University of Innsbruck, Innsbruck, Austria.</span></p>
<p><span style="color: #303030;">The stemness factor Myc and its dimerization partner Max are known to control fundamental cellular processes in vertebrates, in particular with respect to stem cell maintenance and tumor formation. However, their ancestral origins and evolutionary dynamics remain poorly understood. In the present study, Myc and Max conservation across all metazoans will be shown, revealing unexpected cases of gains and losses of </span><span style="color: #303030;"><em>myc</em></span><span style="color: #303030;"> genes at the phylum level. Vertebrates diversified into three </span><span style="color: #303030;"><em>myc</em></span><span style="color: #303030;"> orthologs commonly known to encode for c-Myc, L-Myc and N-Myc. In comparison, cnidarians diversified more extensively into four to six Myc proteins, including a Myc protein lacking all the common N-terminal domain. This unusual Myc isoform is present in Hydra and has been named Myc3. Expression analysis using in situ hybridization and single-cell transcriptome sequencing show that the </span><span style="color: #303030;"><em>Hydra myc3</em></span><span style="color: #303030;"> gene is specifically activated in early to late interstitial precursor cells committed for nerve and gland cell differentiation. To see whether principal functions of this structurally unique </span><span style="color: #303030;"><em>Hydra</em></span><span style="color: #303030;"> Myc are already associated with vertebrate Myc, we performed cell transformation assays in avian cells. Unexpectedly, the conserved C-terminal domain exhibited very high oncogenic capacities. These oncogenic properties, aspects of transcriptional activation and protein-protein interaction will be discussed in relation to 3D structure modelling. Based on these data, we propose that Myc3 may act as a dominant negative factor competing with the stemness maintaining factors Myc1 and Myc2 for dimerization with their natural partner Max.</span></p>
<p><span style="color: #303030;">Hartl, M. et al. (2019). Differential regulation of myc homologs by Wnt/&beta;-Catenin signaling in the&nbsp;</span></p>
<p><span style="color: #303030;">early metazoan Hydra. The FEBS Journal, 286(12), 2295-2310.</span></p>
<p><span style="color: #303030;">This research was supported by the EC H2020 Marie Sklodowska-Curie COFUND research grant No 847681 &ldquo;ARDRE&mdash;Ageing, Regeneration and Drug Research&rdquo;.</span></p>
<hr /><p><strong>Interactions between symbiosis, innate immunity, and nutrition in cnidarians</strong></p>
<p>Maria Valadez Ingersoll, Pablo J. Aguirre Carri&oacute;n, Niharika A. Desai, Caoimhe Bodnar, Sarah W. Davies, Thomas D. Gilmore</p>
<p>Boston University; Cellular and Molecular Biology; Boston, Massachusetts, 02215</p>
<p>Coral reefs are under immediate threat from multiple environmental stressors; understanding the molecular mechanisms governing symbiosis is key to successful restoration efforts. The mutualistic symbiosis between certain cnidarian species and photosynthetic algae of the family Symbiodiniaceae modulates the host&rsquo;s immune and stress responses. I investigate the role of the innate immune system protein Nuclear Factor kB (NF-kB) as an indicator of symbiosis and stress response programming. Starvation of the facultatively symbiotic sea anemone <em>Exaiptasia pallida </em>(Aiptasia) leads to increased NF-B protein levels and gene pathways, but reduced expression of antimicrobial and stress response pathways, regardless of symbiotic state. Symbiont cell density also decreases over one month of starvation, indicating starvation as a stimulus for bleaching. Finally, we performed single-cell RNA sequencing (scRNA-seq) on aposymbiotic and symbiotic branches of the facultatively symbiotic coral <em>Oculina arbuscula. </em>These resultsreveal distinct transcriptional patterns in a subset of 18 projected cell subtypes. Future scRNA-seq of facultative corals across stress conditions will reveal cellular-scale dynamics of gene expression. This research suggests a regulatory role of the cnidarian immune system under certain stressors and implies that symbiotic cnidarians have two pathways for immune regulation &ndash; one to control symbiosis and one to defend against pathogen infection. The latter system may be downregulated when resources are limited.</p>
<p>Mansfield KM, Gilmore TD (2019). Innate Immunity and cnidarian-Symbiodiniaceae &nbsp; &nbsp;</p>
<p>mutualism. <em>Dev Comp Immunol </em>90:199-209.</p>
<p>MVI was supported by an NSF GRFP and NSF NRT DGE 1735087. MVI and PJAC were supported by National Science Foundation NSF IOS-SDS1937650 awarded to SWD and TDG. NAD and CB were supported by the Boston University Undergraduate Research Opportunities Program.</p>
<hr /><p><strong><em>Hydractinia</em></strong><strong> Genome stability</strong></p>
<p><span style="text-decoration: underline;">Helen R Horkan</span><span style="text-decoration: underline;">1</span>*, Noel F Lowndes1, Jordi Solana2, Uri Frank1.</p>
<p>1Centre for Chromosome Biology, School of Biological and Chemical Sciences, University of Galway, Ireland.</p>
<p>2Department of Biological and Medical Sciences, Oxford Brookes University, Oxford, UK.</p>
<p><em>Hydractinia</em> is a highly regenerative animal, that shows no signs of age-related deterioration, develops no spontaneous neoplasia, and is highly resistant to ionizing irradiation (IR). These features are thought to depend on a population of adult pluripotent stem cells called i-cells and may indicate the presence of a highly stable genome in some or all <em>Hydractinia</em> cell types. Our work shows that <em>Hydractinia </em>possesses no unique protection against IR-induced DNA double-strand breaks (DSBs), and that cells clear &gamma;H2A.X foci within 24 hours, in-line with other organisms. However, in contrast to other animals, <em>Hydractinia</em>&nbsp; stem cells are not sensitive to IR. Following irradiation, cycling i-cells exit the cell cycle for up to 9 days. The animals continue growing, which appears to depend on i-cell migration and differentiation only. We hypothesize that i-cells possess a novel mechanism for the maintenance of a stable genome. To address this hypothesis, acetic acid methanol (ACME) whole animal maceration followed by split pool ligation barcoding (SPLiT) was used to generate single-cell RNA seq libraries from irradiated feeding polyps at 1 and 9 days post exposure to 50 Gy gamma irradiation. Using these libraries, we will investigate cell type-specific accumulation of mutations and differential gene expression at single cell resolution Understanding these mechanisms could provide insight into cnidarians' overall resilience to age-related degeneration and cancer.</p>
<p>Garc&iacute;a-Castro, H., et al. (2021). ACME dissociation: a versatile cell fixation-dissociation method&nbsp;</p>
<p>for single-cell transcriptomics. Genome Biology.</p>
<p>*Presenting author HRH is funded by Science Foundation Ireland Centre for Research Training in Genomic Data Science (18/CRT/6214).</p>
<hr /><p><strong>Origin of metazoan and origin of jellyfish; genetic and cell-biology approaches.</strong></p>
<p><span style="text-decoration: underline;">Tsuyoshi Momose</span><span style="text-decoration: underline;">1</span><strong>,</strong> Julie Uveira1, Evelyn Houliston1, Antoine Donati2, Marvin Leria13, Marion Lechable14 and Brandon Weissbourd5</p>
<p>1 LBDV, IMEV, CNRS/Sorbonne Universit&eacute;, Villefranche-sur-Mer, France</p>
<p>2 Biologie du D&eacute;veloppement, CNRS/Inserm/Sorbonne Universit&eacute;, Paris, France</p>
<p>3 IDBM, CNRS/Universit&eacute; de Marseille, Marseille, France</p>
<p>4 University of Innsbruck, Institute of Zoology, Austria</p>
<p>5 Caltech, Pasadena, USA</p>
<p>We study the mechanisms of cnidarian developmental processes to answer questions about early metazoan evolution. We selected <em>Clytia hemisphaerica</em> as a model animal, appreciating its transparent body suitable for live imaging and vegetative growth of the polyp stage, allowing colonial maintenance of genetic strains. In this talk, we summarise our recent hypothesis raised from cell biology studies in <em>Clytia</em> concerning the origin of the metazoan body axis and the jellyfish body plan.</p>
<p>First, we will shortly talk about cell polarity-driven body axis organisation, which provides insights into the body axis symmetry break in the metazoan ancestors.</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
<p>We will also talk about the germ layer origins of the jellyfish bodies. By cell-lineage tracking with Tol2-mediated transgenesis (Weissbourd 2021), we revealed that endoderm (gastrodermis) and ectoderm (epidermis) germ layers were fully segregated by the polyp stage, like in <em>Hydra</em>. Interestingly, endoderm lineage gave rise to the sub-umbrella structures in the jellyfish stage, both epidermal muscle and gastrodermis layers, while ectoderm lineage formed ex-umbrella and tentacle epidermis. Entocodon is formed from the polyp endoderm layer during medusa bud formation. Considering the jellyfish and polyp body plan studied by gene expression (Kraus 2015), a large-scale &ldquo;heterotopic shift&rdquo; (Haeckel 1873) of the axial development program between germ layers occurred when the medusa stage was innovated. Cellular and genetic mechanisms behind the formation of the entocodon are essential for understanding the innovation of the jellyfish stage in the Hydrozoan ancestor.</p>

### Saturday, September 10

#### Keynote

<h3 style="text-align: center;">Keynote</h3>
<p><strong>Regeneration and development: the acoelomorph perspective</strong></p>
<p>Mansi Strivastava</p>
<p>Harvard University</p>
<p>Dr. Srivastava received her A.B. in Biological Sciences from Mount Holyoke College, where she became fascinated by the process of regeneration and wrote her honors thesis on regeneration in segmented worms. She studied animal evolution using comparative genomics for her Ph.D. in Molecular and Cell Biology from the University of California at Berkeley. Her thesis research included work on two cnidarians, <em>Nematostella</em> and <em>Hydra</em>. For her postdoctoral training at the Whitehead Institute/MIT, Dr. Srivastava returned to her interest in regeneration and developed the acoel <em>Hofstenia miamia</em>, a.k.a. the three-banded panther, worm as a new research organism for studying the evolution of regeneration. In 2015, Dr. Srivastava joined the faculty of Organismic and Evolutionary Biology at Harvard University and became a Curator of Invertebrate Zoology at the Museum of Comparative Zoology. Her research group uses panther worms to develop new approaches for studying both the mechanisms and evolution of regeneration.</p>

#### Session 10: Development II

<hr /><p><strong>A post-larval stem-like cell population contributes to germinal and somatic lineages in the sea anemone </strong><strong><em>Nematostella vectensis</em></strong></p>
<p><span style="text-decoration: underline;">Paula Miram&oacute;n-Pu&eacute;rtolas</span>1, Marion Lebouvier1, Marie Montjourid&egrave;s1, Yann Loe-Mie2, Heather Marlow2, Patrick Steinmetz1</p>
<p>1Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway</p>
<p>2(Epi)genomics of Animal Development, Department of Developmental and Stem Cell Biology, Institut Pasteur, Paris, France</p>
<p>The separation of germline and soma during embryogenesis has traditionally been considered a key step during animal development. Post-embryonic germline segregation, however, has been described in an increasing number of animals (e.g. planarians, sponges or hydrozoan cnidarians). This raises the question if post-embryonic germline segregation may have been an ancient animal feature, yet the lack of molecular and developmental data available for non-bilaterians hampers our understanding of germline evolution. Here, we aimed to identify and characterize the germline stem cell lineage in the sea anemone <em>Nematostella vectensis</em>. We have discovered a population of proliferative, stem-like cells in extra-gonadal regions of juvenile and adult mesenteries that express the conserved germline/stem cell marker gene orthologs <em>vasa</em> and <em>piwi</em>. <em>vasa2</em> and <em>piwi1 </em>transgenic reporter lines validate the location of stem-like cells and their germinal progeny (oocytes and spermatogonia). Strikingly, fate mapping of the stem-like cells reveals diverse populations of proliferative somatic cells, some with neuron-like shapes, throughout the body of both juveniles and adults. Using a <em>soxB2 </em>reporter line, we confirm that a subset of these cells consists of neural progenitors. These results indicate that the <em>vasa2+/piwi1+</em> stem-like cell population may represent a post-embryonic, multipotent stem cell population in <em>Nematostella vectensis</em> that gives rise to both germinal and neuronal lineages. As these stem-like cells are reminiscent of the interstitial stem cells (i-cells) of hydrozoan cnidarians, we propose that the presence of adult multipotent stem cells and post-embryonic germline segregation may be evolutionarily conserved in cnidarians.</p>
<hr /><p><strong>Functional characterization of canonical Wnt receptors and ligands required for oral-aboral patterning in </strong><strong><em>Nematostella vectensis</em></strong></p>
<p><span style="text-decoration: underline;">Fredrik Hugosson</span>, Camille Enjolras, Brent Foster and Mark Q Martindale</p>
<p>Whitney laboratory for marine bioscience. University of Florida.</p>
<p>The canonical Wnt (cWnt) signaling pathway is well described in Nematostella. From pioneering work, cloning and resolving expression patterns for genes in the pathway to analysis of the effect of ectopic activation using GSK-3b inhibitors. From this we know that cWnt signaling is required for endomesoderm formation and Oral-Aboral (O-A) axial patterning of the ectoderm. Still most cWnt components such as ligands and receptors have not been functionally characterized in Nematostella. Therefore it is still unclear what independent function they have. Is there redundant or unique functions for each of them? We have focused our analysis on O-A patterning of the embryo using beta-catenin dependent genes, oral marker Brachyury (Bra), midbody marker Wnt2 and aboral marker Six3/6.</p>
<p>Considering the LRP5/6 receptor to be a core component of cWnt signaling we expected knockdown (KD) of the single LRP5/6 to alone show cWnt dependent phenotypes. The morphological phenotype from LRP5/6 KD is very similar to expression of dominant negative Tcf where gastrulation occur but development of oral structures are lost. KD of LRP5/6 results in loss of Bra expression, reduced Wnt2 and expanded Six3/6 expression. This without effecting the beta-catenin dependent specification of endomesoderm which likely is regulated by maternal components. Next we analyzed the role of Frizzled receptors. To phenocopy the results of LRP5/6 KD we had to KD all four Fz genes together. Suggesting that all four Fz together with LRP5/6 is required for O-A patterning. None of the Wnts alone effect the O-A patterning like LRP5/6 KD but when we combine KD of Wnt3 and 4 we recapitulate the phenotype. We conclude that Wnt3 and Wnt4 together with LRP5/6 and Fz are required for O-A patterning. Our results are in agreement with a recent study by Niedermoser et al.</p>
<p>This work is supported by NSF grant #1755364</p>
<p>Niedermoser et al. 2022 <em>bioRxiv</em> 2022.03.15.484449</p>
<hr /><p><strong>An FGF activity gradient signals through a cnidarian paired-like homeodomain gene to direct apical sensory organ development in </strong><strong><em>Nematostella vectensis</em></strong></p>
<p><span style="text-decoration: underline;">Keith Z. Sabin</span><span style="text-decoration: underline;">1</span>, Eric Hill1, Shiyuan Chen1, Mathew C. Gibson1</p>
<p>1Stowers Institute for Medical Research, Kansas City, MO, USA</p>
<p>Sensory organs generally consist of a few functionally and molecularly distinct cell types, including specialized sensory cells, non-sensory support cells, and dedicated sensory neurons. The apical sensory organs of marine invertebrate larvae represent a conserved and prototypic sensory system that predates nervous system centralization. However, the cellular and molecular heterogeneity within these larval sensory structures remains unclear. Here, we use <em>Nematostella vectensis</em> as a genetically tractable system to interrogate the cell type composition and gene regulatory network controlling apical organ development. Using single cell RNA-seq, we found that the <em>Nematostella</em> apical sensory organ is comprised of two unique cell types: GABAergic sensory cells and a putative non-sensory support cell population. Interestingly, an FGF activity gradient regulates the specification of both sensory cells (high FGF) and support cells (lower FGF). High FGF signaling conditions induce the expression of a paired-like homeodomain gene, <em>prd146</em>, which functions to promote sensory cell specification and inhibit support cell specification. Using epistasis experiments, we place <em>prd146</em> in the apical sensory organ gene regulatory network and identified putative PRD146 target genes. Finally, using CRISPR mutagenesis we investigate the function of the apical sensory organ during planula metamorphosis. Collectively, these experiments revealed an unanticipated degree of cellular and molecular complexity in a prototypic sensory structure.</p>
<p>Funding: Stowers Institutional Funds</p>
<p>Rentzsch, et al. FGF signalling controls formation of the apical sensory organ in the cnidarian Nematostella vectensis. Development 135, 1761&ndash;1769 (2008).</p>
<p>Sinigaglia, et al. The bilaterian head patterning gene six3/6 controls aboral domain development in a cnidarian. PLoS Biol. 11, e1001488 (2013).</p>
<hr /><p><strong>FGF signaling in the freshwater polyp </strong><strong><em>Hydra</em></strong></p>
<p>Kerstin Ohler, Lisa Reichart, Annegret Rittershaus, Ellen Lange, and Monika Hassel</p>
<p>Philipps-Universit&auml;t Marburg, Marburg, Germany</p>
<p>During eumetazoan development, the fibroblast growth factor receptor (FGFR) signaling pathway is essential for patterning and morphogenesis, regulating key aspects of cellular physiology such as proliferation, differentiation, and cell migration (1, 2).</p>
<p>In <em>Hydra</em>, a simple freshwater polyp belonging to the ancestral phylum Cnidaria, the FGFR pathway is already present. Two FGFRs (3, 4), and five potential ligands (5) were recently identified.</p>
<p>FGFRa is essential for the detachment of vegetative buds (6, 7). And according to our unpublished data, it seems to have an additional function in the selective guidance of cells along the body column. I-cell migration towards regions of stronger FGF transcription is prevented upon local or systemic inhibition of FGFR.</p>
<p>The five potential FGFR ligands show each a characteristic expression pattern that we visualized by <em>in situ</em> hybridization. On the basis of their differential expression and siRNA knockdown experiments, we propose a model, in which <em>Hydra</em> FGFs selectively guide already committed interstitial cells and/or serve as differentiation signals, while a third truncated FGFR might act as an FGF scavenger in the tentacle region to slow down cells on their way towards the hypostome.</p>
<p>To confirm this model, it is important to gain more conclusive evidence than manipulations on the pharmacological and RNA levels can provide. Therefore, we are currently using purified recombinant <em>Hydra</em> FGF proteins to investigate their effects on cell differentiation and migration <em>in vitro</em> and<em> in vivo</em>. Our goal is, to gain further insight into the functions of FGF/R signaling in <em>Hydra</em> and to learn about the evolution of FGF/R signaling in Eumetazoa.</p>
<p>(1) Ornitz and Itoh, 2015; (2) Thisse and Thisse, 2005; (3) Sudhop et al., 2004; (4) Suryawanshi et al., 2020; (5) Lange et al., 2014; (6) Hasse et al., 2010; (7) Holz et al., 2020</p>
<p>This research was funded by DFG HA-1732/8, HA-1732/13</p>
<hr /><p><strong>The Hippo pathway and </strong><strong><em>Hydra</em></strong><strong> morphogenesis.</strong></p>
<p><span style="text-decoration: underline;">Maria Brooun</span>1*, <span style="color: #201f1e;">Willi Salvenmoser</span><span style="color: #201f1e;">2</span><span style="color: #201f1e;">, Catherine Dana</span><span style="color: #201f1e;">3</span><span style="color: #201f1e;">, Marius Sudol</span><span style="color: #201f1e;">4</span><span style="color: #201f1e;">, Robert Steele</span><span style="color: #201f1e;">3</span><span style="color: #201f1e;">, Bert Hobmayer</span><span style="color: #201f1e;">2</span><span style="color: #201f1e;">, Helen McNeill</span><span style="color: #201f1e;">5*</span></p>
<p><span style="color: #201f1e;">1</span><span style="color: #201f1e;"> Lunenfeld-Tanenbaum Research Institute, Toronto, Canada.</span></p>
<p>2Department of Zoology and Center for Molecular Biosciences Innsbruck, University of Innsbruck, Austria.</p>
<p>3 Department of Biological Chemistry, School of Medicine, University of California, Irvine, USA.</p>
<p>4 Icahn School of Medicine at Mount Sinai, Department of Medicine<span style="color: #201f1e;">, </span>New York, USA.&nbsp;</p>
<p>5Department of Developmental Biology, Washington University School of Medicine, St. Louis, USA.</p>
<p>The Hippo pathway is a key regulator of cell proliferation, differentiation, and apoptosis, and is remarkably conserved in metazoans and their unicellular ancestors. It consists of a cascade of kinases that controls nuclear localization of the transcriptional co-activator Yap. We raised an antibody against the <em>Hydra</em> homologue of Yap, HyYap, and showed that its accumulation in the nuclei of ectodermal epithelial cells depends on the axial position: it is the strongest in the peduncle and tentacles, and the weakest in the budding zone. Using <em>shRNA</em>-mediated knockdown as well as a gain of function transgenic approach, we show that HyYap negatively regulates budding. In contrast, the <em>Hydra</em> homologue of Lats kinase, a negative regulator of Yap, is required for budding. Our epistasis experiments show that during budding the Hippo pathway operates upstream of canonical Wnt signaling, a well-established regulator of bud initiation. We also show a role for the Hippo pathway in <em>Hydra</em> morphogenesis: inhibition of Lats kinase dramatically affects the actin cytoskeleton and epithelial cell morphology, and leads to formation of shortened tentacles. We demonstrate that, similar to <em>Drosophila</em> and mammals, in <em>Hydra</em> the amount of nuclear Yap is reduced in the area of high cell density, the budding zone. We speculate that the link between Hippo signaling and continuous cell division, cell density, and axis formation arose early in metazoan evolution.</p>

#### Session 11: Evo Devo II

<hr /><p><strong>Genomic consequences of the loss of the polyp stage in the scyphozoan </strong><strong><em>Pelagia</em></strong></p>
<p>Manon Boosten 1,2, Camille Sant 1, Richard Copley 1, Sami El Hilali 1, Jean-Marc Aury 3, Yulia Kraus 4, Lionel Guidi 2, <span style="text-decoration: underline;">Lucas Lecl&egrave;re</span>1,5</p>
<p>1 Sorbonne Universit&eacute;, CNRS, Laboratoire de Biologie du D&eacute;veloppement de Villefranche-sur-mer (LBDV), France.</p>
<p>2 Sorbonne Universit&eacute;, CNRS, Laboratoire d&rsquo;Oc&eacute;anographie de Villefranche-sur-mer (LOV), France.</p>
<p>3 Genoscope, Institut Fran&ccedil;ois Jacob, CEA, CNRS, Univ Evry, Universit&eacute; Paris-Saclay, Evry, France.</p>
<p>4 Department of Evolutionary Biology, Faculty of Biology, Lomonosov Moscow State University, Moscow, Russia.</p>
<p>5 Sorbonne University, CNRS, Laboratory of Integrative Biology of Marine Organisms, Banyuls-sur-Mer, France.</p>
<p>Medusozoan cnidarians exhibit a wide variety of shapes, developmental strategies, and life cycles. Many species harbor a complex life cycle, which features a planula larva metamorphosing into a benthic polyp stage, which then asexually produces large numbers of free-swimming clonal medusae. During the course of evolution, some species lost the polyp stage and acquired a simplified life cycle in which neither the embryonic or adult stages settle on a substrate. The scyphozoan <em>Pelagia noctiluca</em> displays such a reduced life cycle, where the swimming planula develops directly into a juvenile medusa, called ephyra. To address the genomic, developmental and ecological consequences of the loss of a life stage, we developed <em>Pelagia</em><em>noctiluca</em> as a new, tractable model system. We have generated extensive genomic and transcriptomic resources and investigated the temporal and spatial dynamics of key cnidarian developmental regulators during <em>Pelagia </em>embryogenesis and ephyra development. The comparison of <em>Pelagia</em> data with closely related species harboring a polyp stage offers insights into the developmental mechanisms acting during scyphozoan embryogenesis and the sexually-derived medusa, as well as into the genomic consequences of life cycle simplification in medusozoans.</p>
<p>This work is supported by: Agence Nationale pour la Recherche (ANR), Centre National pour la Recherche Scientifique (CNRS).</p>
<hr /><p><strong>Origins of individuality in colonial siphonophores&nbsp;&nbsp;</strong></p>
<p><span style="text-decoration: underline;">Maciej Mańko</span>1, Catriona Munro2,3, Lucas Lecl&egrave;re3,4</p>
<p>1University of Gdańsk, Gdańsk, Poland</p>
<p>2Coll&egrave;ge de France, Paris, France</p>
<p>3Sorbonne Universit&eacute;/CNRS, Villefranche-sur-mer, France</p>
<p>4Sorbonne Universit&eacute;/CNRS, Banyuls-sur-Mer, France</p>
<p>Coloniality is a widespread trait among marine invertebrates that offers many ecological advantages. Colonies exemplify varying degrees of physiological integration and functional specialization of colony building blocks (zooids), with siphonophores representing the most complex type of colony-level organization. In siphonophores, zooids are budded off in a fixed pattern, resulting in iterative, species-specific zooid clusters called cormidia. In some species, posterior-most cormidia are sequentially detached from the adult colonies, which disperse, develop gonads, and reproduce sexually. These free-living, self-sufficient colony fragments (eudoxids), bear hallmarks of individuality, yet their formation, evolution or even ecology were so far not investigated. Here we tackled this issue with an integrative eco-evo-devo approach. Using time lapse movies, immunohistochemistry and pharmacological inhibition we provide mechanistic understanding of eudoxid formation. Upon reaching certain morphological threshold cormida are released by constriction of a detachment ring, a wide band of muscle fibers surrounding colony stem. Released cormidia acquire positive buoyancy through mesoglea remodeling and volume increase of one the cormidial zooids, and then swims away as a newly formed eudoxid. We found that eudoxid release evolved once in monophyletic Diphyomorpha, concomitantly to the acquisition of the detachment ring. We also documented newly formed eudoxids mimicking feeding behavior of polygastric colonies, and found evidence of spatial and temporal ecological niche partitioning between the two. Overall, our study uncovers a unique life history transition induced by the acquisition of a novel muscle, leading to the evolution of an individual-like dispersive stage in the life cycle of colonial siphonophores.</p>
<hr /><p><strong>Coevolution of the </strong><strong><em>Tlx </em></strong><strong>homeobox gene with medusa development (Cnidaria: Medusozoa)</strong></p>
<p>Matthew Travert1, Reed Boohar1*, Steven M. Sanders2, Matthew L. Nicotra2 Lucas Lecl&egrave;re3,</p>
<p>Robert E. Steele4, Paulyn Cartwright1</p>
<p>1University of Kansas,</p>
<p>2University of Pittsburgh,</p>
<p>3Sorbonne Universit&eacute;, CNRS, Laboratoire de Biologie du D&eacute;veloppement de Villefranche-sur-Mer (LBDV), Villefranche-sur-Mer, France,</p>
<p>4Department of Biological Chemistry, University of California, Irvine</p>
<p>*Current affiliation: Department of Biology, University of Miami, Coral Gables, Florida</p>
<p>Medusozoans are a group of cnidarians characterized by the presence of a medusa (jellyfish) stage as part of the life cycle. Using genomes, transcriptomes, and degenerate PCR, from species distributed across the phylum Cnidaria, we found that the presence of the homeobox gene <em>Tlx</em> is correlated with those species that have a medusa stage as part of the life cycle. Although the homeobox gene <em>Tlx</em> is highly conserved across metazoans it is lacking in the genomes of anthozoans (corals and sea anemones) and endocnidozoans (parasitic <em>Polypodium</em> and myxozoans). Phylogenetic analyses reveal that medusae have been lost or reduced at least 15 times in the medusozoan class Hydrozoa and Bayesian correlation analysis reveals that loss of the <em>Tlx</em> gene is highly correlated this pattern of medusae loss. In instances where <span style="text-decoration: underline;"><em>Tlx</em></span> could be detected in species that lack a medusa stage, these were revealed to be pseudogenes and/or undergo relaxed selection. In addition, selection analyses reveal that hydrozoans in general undergo relatively relaxed selection of <span style="text-decoration: underline;"><em>Tlx</em></span>compared to the other medusozoan classes, which may in part explain their pattern of multiple medusa losses.&nbsp; These data, together with our gene expression analyses, suggest an evolutionary link between loss of a single gene and loss of a complex trait.</p>
<hr /><p><strong>Yolk formation in a sea anemone provides insights into the evolution of animal nutrient transport</strong></p>
<p>Marion Lebouvier1, Paula Miram&oacute;n-Pu&eacute;rtolas1, <span style="text-decoration: underline;">Patrick R. H. Steinmetz</span><span style="text-decoration: underline;">1</span></p>
<p>1Sars Centre, University of Bergen, Thormoehlensgt. 55, N-5008 Bergen, Norway</p>
<p>Circulatory systems play an important role in many bilaterian animals (e.g. vertebrates, flies, annelids) to transport dietary nutrients towards oocytes during yolk formation. Currently, however, only little is known about the evolutionary origin of nutrient transport systems in animals. Here, we have characterized dietary nutrient transport during vitellogenesis in the sea anemone <em>Nematostella vectensis</em> (Anthozoa, Cnidaria) as a paradigm to study the evolution of animal nutrient distribution systems. Using a combination of fluorescent bead uptake and expression analysis of key marker genes, we found that the gonad epithelium exhibits increased levels of phagocytosis, micropinocytosis and intracellular digestion of food components. Pulse-chase experiments further show that labelled fatty acids rapidly translocate from the gonad epithelium through the extracellular matrix (ECM) into oocytes. Expression of conserved lipid transport proteins <em>vitellogenin </em>(<em>vtg</em>) and <em>apolipoprotein-B</em> (<em>apoB</em>), and colocalization of labelled fatty acids with an endogenously tagged ApoB-PSmOrange apolipoprotein in the gonad epithelium further support the lipid-shuttling role of the gonad epithelium. In a complementary fashion, we find oocyte expression of <em>very low-density lipoprotein receptor</em> (<em>vldlr</em>) orthologs, conserved in bilaterian Vtg/ApoB-mediated endocytosis of lipids. These findings support that the Vtg ligand/VLDL receptor pair is evolutionary conserved between sea anemone and bilaterians to mediate lipid transport during vitellogenesis. In addition, we identified ECM-based, mesenchymal-like cells with potential role in systemic lipid transport. Altogether, our work supports a long-standing hypothesis that an ECM-based lipid transport system predated the cnidarian-bilaterian split.</p>
<h3 style="text-align: center;">Invited Technology Speaker</h3>

#### Session 12: Regeneration

<p><strong>Live-imaging metabolic states in skin stem cells to reveal adaptations to oncogenic mutations</strong></p>
<p>Anupama Hemalatha</p>
<p>Yale University</p>
<p>Dr. Hemalatha&rsquo;s graduate training was at the intersection of cell biology and developmental biology. During her PhD, she studied how multiple endocytic pathways co-regulate and add nuance to the Wingless- signaling pathway in Drosophila wing discs. This sparked her interest in how intersecting complex growth signaling pathways are deciphered per cell to result in cell behaviors in a tissue-specific coordinated manner. Energy metabolism is one such hub that integrates growth signals and directs cell behaviors like proliferation and differentiation. In her talk, she will highlight the technical advances in imaging the skin of live mice pioneered by the Greco Lab and the adaptations that enable tracking of cellular metabolic state, in the skin stem cells over time. Using optical redox imaging to monitor endogenous fluorescence of metabolites NAD(P)H and FAD, she has observed early changes in the redox state of skin stem cell layer in the presence of oncogenic mutations. This imaging modality enables her to track the subsequent metabolic adaptations at the mutant-wild-type cell interface as homoeostasis is re-established.</p>
<hr /><p><strong>Progenitor cell invasion during </strong><strong><em>Hydra vulgaris</em></strong><strong> head regeneration</strong></p>
<p>Ben Cox and Celina Juliano</p>
<p>University of California, Davis</p>
<p>Cell invasion through extracellular matrix (ECM) is an essential process in animal development and pathology (e.g. cancer) but has scarcely been studied during regeneration, even though injury creates a need to reconstruct tissue layers. Regeneration employs processes that occur in development but often across compressed time scales and complemented by regeneration-specific behaviors. Thus, cell invasion during regeneration may exhibit unique properties in contrast with its role in development and homeostasis. <em>Hydra</em> vulgaris has many advantages for studying cell invasion&mdash;it regenerates quickly, is amenable to live imaging, and has a simple body plan consisting of two epithelial layers separated by ECM. <em>Hydra</em> have interstitial stem cells (ISCs), which reside in the ectoderm and give rise to cell types that reside only in the endoderm, including gland cells and some neuron cell types; their invasion into endoderm is thus essential for nervous and digestive system function. Further, Hydra passively displace and shed cells from its extremities throughout its life, necessitating ISC lineage cells to invade through the ECM to replace the lost differentiated cells. Endodermal neurons and gland cells are also lost after injury. Upon head amputation, the remaining epithelial layers quickly close the wound. However, principal ECM components laminin and collagen do not travel with wound-closing epithelial cells; thus, ectodermal and endodermal cells are not separated by ECM at the site of injury for about 24 hours. Using a transgenic reporter that marks ISCs, we show that ISC lineage cells originating in the ectoderm migrate into the endoderm at regions of the head lacking or expressing low levels of laminin and collagen. This pattern of migration contrasts with homeostatic growth and budding, in which progenitors invade through intact ECM. Thus, <em>Hydra</em> provides a model to study context-dependent modes of cell invasion and the role of progenitor invasion during regeneration.</p>
<p>Funding: Society for Developmental Biology Emerging Research Organisms Grant</p>
<hr /><p><strong>Distinct stem-like cell populations coordinate tissue elongation and regeneration in </strong><strong><em>Cladonema </em></strong><strong>medusa tentacles</strong></p>
<p><span style="text-decoration: underline;">Sosuke Fujita</span>1, Erina Kuranaga1, Masayuki Miura 2, Yu-ichiro Nakajima2</p>
<p style="text-align: justify;">1Graduate School of Life Sciences, Tohoku University, Sendai, Miyagi, Japan</p>
<p style="text-align: justify;">2Graduate School of Pharmaceutical Sciences, The University of Tokyo, Tokyo, Japan</p>
<p>Cnidarians including hydra, corals, and jellyfish maintain a high regenerative capacity throughout their life cycle. While free-swimming medusae, or jellyfish, exhibit complex morphology compared to sessile polyps, they still have a capacity for body growth and regeneration. Previous studies on cnidarian polyps have suggested that migration multipotent stem cells and blastema formation by multipotent stem cells are involved in regeneration; however, the mechanism of regeneration in medusae is largely unknown. Here we investigate the mechanisms of organ regeneration, using medusa tentacles in the hydrozoan jellyfish<em> Cladonema pacificum</em>. The <em>Cladonema</em> medusa tentacle is a tube-like organ with continuous elongation and branching, consisting of epithelial cells, proximally-localized stem cells, and distally distributed cnidocytes, or &ldquo;stinging cells&rdquo;. We show that proliferative cells accumulate at the wound site during regeneration and form a blastema composed of stem-like cells. Nucleoside-analog chase experiments indicate that proliferative cells in the blastema are distinct from the original stem cells, suggesting a potential fate conversion of the surrounding cells into stem-like cells. We further reveal that the original stem cells rarely migrate while stem-like cells in the blastema differentiate into neurons and cnidocytes, which are involved in re-formation of functional tentacles. These results suggest the existence of a mechanism that maintains the original stem cell population and supplies regeneration-specific stem-like cells, which may efficiently allow tentacle elongation and functional organ regeneration.</p>
<p>Reference</p>
<p>- Fujita S et al., <em>PeerJ</em> 2019</p>
<p>- Fujita S et al., <em>Genes</em> 2021</p>
<hr /><p><strong>Comparative approach for understanding foot regeneration in </strong><strong><em>Hydra</em></strong></p>
<p><span style="text-decoration: underline;">Sergio E. Campos</span>, Craig Ciampa, Jack Cazet &amp; Celina Juliano</p>
<p>University of California, Davis</p>
<p>Some members of the genus <em>Hydra </em>have amazing whole animal regeneration abilities. <em>Hydra&rsquo;s</em> body plan consists of a body column with a head at the oral end and a foot at the aboral end. In <em>H.</em><em>vulgaris</em>, bisection of the body column elicits head regeneration at the oral-facing wound and foot regeneration at the aboral-facing wound. In <em>H. vulgaris,</em> Wnt signaling plays a central role in head regeneration after amputation. However, little is known about the mechanisms that initiate foot regeneration in <em>Hydra. </em>Interestingly, we find that <em>Hydra</em><em>oligactis</em> exhibits a low rate of foot regeneration. This difference in foot regeneration rate between both species lends itself for comparative studies to reveal new principles of regeneration. For instances, an important unanswered question is to understand the injury-induced triggers of foot regeneration. To gain insight into this question, we collected RNA-seq data to characterize the divergence of the transcriptional response to injury between regenerating (<em>H. vulgaris</em>) and non-regenerating (<em>H. oligactis</em>) foot tissue. These data revealed a difference in the timing of transcriptional activation of the Wnt signaling pathway. In <em>H. vulgaris </em>Wnt genes are up-regulated during early injury response in both regenerating head and feet, whereas in <em>H. oligactis</em> this response is delayed. To further understand the difference in Wnt expression between both species, we are using <em>in situ</em> hybridization in <em>H. oligactis</em>.Moreover, to test the hypothesis that early activation of the Wnt pathway is required to initiate foot regeneration we are chemically modulating Wnt activation after foot amputation in <em>H. oligactis</em>, and determining the effect on foot regeneration. We predict that some Wnt activation is necessary for foot regeneration, however, early expression is inhibited in <em>H. oligactis </em>at the aboral-facing injury by a strong Wnt gradient resulting in foot regeneration defects. We anticipate that our results will improve our understanding of how regeneration is triggered in different contexts.</p>
<hr /><p><strong>To regenerate or not to regenerate? Towards an integrative model for the recovery of shapes in damaged </strong><strong><em>Clytia</em></strong><strong> medusae</strong></p>
<p><span style="text-decoration: underline;">Chiara Sinigaglia</span>1; 2, Sophie Peron 1, Lucas Lecl&egrave;re 1; 2</p>
<p>1 Sorbonne University/CNRS, Laboratory of Developmental Biology of Villefranche-sur-Mer, France</p>
<p>2 Sorbonne University/CNRS, Laboratory of Integrative Biology of Marine Organisms, Banyuls-sur-Mer, France</p>
<p>How randomly injured animals can appropriately re-establish positional information and control the deployment of repair programs are key questions of regenerative biology. The small hydrozoan medusae <em>Clytia hemisphaerica</em>, which are frequently damaged in the plankton, show powerful regenerative capacities, being able to regain a circular shape in less than 12 hours and a new functional mouth in 4 days. This efficient recovery depends on an interplay between mechanical forces, cell migration and proliferation, which we are just starting to unravel. In particular, we showed that the umbrella remodeling causes the radial muscle fibers in the subumbrellar layer to converge into &lsquo;hubs&rsquo;, associated to activation of Wnt signaling, and which function as positional landmarks. The different observed configurations of these muscle fibers correlate with a specific pattern of Wnt signaling activation, and - most remarkably - with the fate of the wound, notably whether a mouth regenerative program will be activated. In a second phase, mouth morphogenesis is fueled by both local cell proliferation and long-range cell recruitment and is further modulated by its connections with the gastrovascular canal system. <em>Clytia</em> medusae offer a novel experimental paradigm for addressing patterning mechanisms and morphogenesis in tractable adult bodies and for dissecting the interplay between chemical and mechanical cues in pattern formation.</p>
<p>Chiara Sinigaglia, Sophie Peron, Jeanne Eichelbrenner, Sandra Chevalier, Julia Steger, Carine Barreau, Evelyn Houliston, Lucas Lecl&egrave;re. (2020) Pattern regulation in a regenerating jellyfish. eLife 9:e54868.</p>
<p>This work is supported by: ATIP-Avenir, Agence Nationale pour la Recherche (ANR), Centre National pour la Recherche Scientifique (CNRS).</p>
<hr /><p><strong>Senescence-induced cellular reprogramming drives whole-body regeneration</strong></p>
<p><span style="text-decoration: underline;">Miguel Salinas-Saavedra</span><span style="text-decoration: underline;">1</span>, Uri Frank1</p>
<p>1National University of Ireland, Galway, Galway, Ireland</p>
<p>In the cnidarian <em>Hydractinia symbiolongicarpus</em> adult, intact polyps, Piwi1+ stem cells (known as i-cells) restrict to the lower body column but are absent from the head. While studying regeneration in <em>Hydractinia</em>, we have discovered that small pieces of tissues from the animal's head can regenerate a fully functional individual. This is surprising, given that the <em>Hydractinia</em> head does not contain any stem cells that are normally required for regeneration in this animal. We identify senescence to be the main factor underlying this phenomenon. Cells at the site of injury in the amputated head transiently display cellular senescence features: cell cycle arrest, CDK inhibitor upregulation, and senescence-associated &szlig;-galactosidase activity that last for 24 hours. Using an array of advanced techniques, including transcriptomics, transgenesis, optogenetics, CRISPR-Cas9 mutagenesis, and <em>in vivo </em>imaging, we experimentally modified the senescence signal triggered by amputation. Our main findings are that amputation injury induces senescence in adjacent cells that are then expelled from the tissue. We show that senescent signalling induces differentiated cells to reprogram and become stem cells that allow for whole-body regeneration. Inhibition of senescence, either pharmacologically or genetically, prevented cellular reprogramming and regeneration, while ectopic induction of senescence in regenerative context-induced supernumerary stem cells and faster regeneration. Recent studies suggest that regenerative senescence is shared between animals as distantly related as vertebrates and cnidarians. Therefore, senescence signalling may be an ancient mechanism mediating cellular plasticity. Understanding the senescence environment that promotes cellular reprogramming could provide a new avenue to understanding animal regeneration and cell-fate specification.</p>

## Poster Abstracts

<p style="text-align: center;"><strong>(Alphabetical by presenter&rsquo;s last name)</strong></p>
<p><strong>Siphonophore genome attributes</strong></p>
<p>Namrata Ahuja1, Darrin Schultz2, Natasha Picciani1, Casey Dunn1</p>
<p>1Yale University, New Haven, USA</p>
<p>2University of Vienna, Vienna, Austria</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Siphonophores are remarkable among animals in the open ocean for their unique colony-level body plans, which highlight functional specialization. Such body plans are comprised of multiple individual &ldquo;bodies&rdquo; or zooids, each with a distinct function yet physiologically integrated. Elucidating the genes and other factors that drive zooid specialization has been challenging. To date there are 190 siphonophore species, which are abundant in all the oceans and play a significant role in the deep-sea jelly web. Despite the novel characteristics siphonophores offer as study organisms and their importance in the open ocean ecosystem, understanding their genomic attributes has been difficult. This has largely been because 1) they are hard to collect, 2) their genomes are large, and 3) until recently, they could not be reared in the lab through their lifecycle. With the advent of deep-sea submersibles, siphonophores have become easier to collect. In order to estimate genome sizes, we sequenced 300-400 million reads of genomic DNA from each of 30 siphonophore species with paired end 150bp Illumina sequencing. K-mer analyses reveal that most siphonophores have very large genomes greater than at least 2 Gb, assuming that 20x coverage is needed for an estimate. We were able to estimate the size of the <em>Cordagalma ordinatum </em>genomes at 700Mb, far smaller than other examined siphonophore genomes. In parallel to the genome skimming, we also sequenced the genome of <em>Nanomia bijuga,</em> which is well positioned as a study organism since it is one of the easier to collect siphonophores, has been successfully cultured at the Monterey Bay Aquarium, and is among the best studied siphonophores. Dovetail produced a well resolved chromosome-scale diploid assembly with high contiguity and completeness using PacBio Hi-Fi sequencing and Omni-C scaffolding. The haploid genome size is 1.7 Gb, placing it among the smaller siphonophore genomes.</p>
<p>This work is funded by NSF-OCE 1829835.</p>
<hr /><p><strong>Investigating the role of Notch in generating mechanosensory hair cells at life cycle transition in the sea anemone </strong><strong><em>Nematostella vectensis</em></strong></p>
<p><span style="text-decoration: underline;">Ndotimi Apulu</span>, Nagayasu Nakanishi</p>
<p>University of Arkansas</p>
<p>Developmental genetic mechanisms that underlie the formation of new cell types during cnidarian life cycle transitions are poorly understood. In Cnidaria, mechanosensory neurons referred to as the concentric hair cells initiate development at life cycle transitions and populate the ectodermal epithelium of polyp tentacles and medusa sensory structures; they are lacking in planula larvae. We have recently demonstrated the role of the class IV POU homeodomain transcription factor in hair cell maturation in the sea anemone cnidarian <em>Nematostella vectensis</em>. However, signaling molecules involved in the initial differentiation of cnidarian hair cells are not known. As Notch signaling plays a pivotal role in mechanosensory development in vertebrates, here we examine if Notch similarly has a role in the initial differentiation of mechanosensory hair cells in <em>N. vectensis. </em>Using transgenic fluorescent reporter assays and antibody staining, we show that Notch is expressed in clusters of epithelial cells in circum-oral ectoderm at the late planula stage, and in scattered ectodermal epithelial cells of developing tentacles - excluding hair cells - at the tentacle-bud and primary polyp stages. In addition, we find that pharmacological perturbation of Notch signaling by gamma-secretase inhibitor DAPT during planula development leads to a precocious increase in hair cell number. These results suggest that Notch signaling negatively regulates the development of cnidarian hair cells and has a role in controlling the timing of hair cell development &ndash; to initiate at life cycle transition.</p>
<p>This research was funded by National Science Foundation.</p>
<p>Ozment, et. al., (2021). Cnidarian hair cell development illuminates an ancient role for the class IV POU transcription factor in defining mechanoreceptor identity. <em>Elife</em>, <em>10</em>, e74336.</p>
<hr /><p><strong>Role of TMC ion channels in the Jellyfish stinging response.</strong></p>
<p><span style="text-decoration: underline;">Zeeshan Z. Banday</span>1,2, Zhen Tong3, Emily Watto1,2, Jocelyn E. Malamy1 and Eduardo Perozo2,3,4</p>
<p><span style="color: #222222;">1</span><span style="color: #222222;">Department of Molecular Genetics and Cell Biology, The University of Chicago, Chicago, IL, USA</span></p>
<p><span style="color: #222222;">2</span><span style="color: #222222;">Department of Biochemistry and Molecular Biology, The University of Chicago, Chicago, IL, USA</span></p>
<p><span style="color: #222222;">3</span><span style="color: #222222;">Pritzker School of Molecular Engineering, The University of Chicago, Chicago, IL, USA</span></p>
<p><span style="color: #222222;">4</span><span style="color: #222222;">Grossman Institute for Neuroscience, Quantitative Biology and Human Behavior, The University of Chicago, Chicago, IL, USA</span></p>
<p>The conversion of mechanical stimuli to electrical signals is of fundamental importance in a variety of physiological responses, including auditory and vestibular function. In the vertebrate inner ear hair cells, a mechanoelectrical transduction (MET) complex is located at the tips of stereocilia. Transmembrane Channel-like 1 and 2 (TMC-1/2) proteins likely function as the mechanosensitive channels in this complex. Characterization of the MET complex in vertebrate hair cells has been a major challenge, primarily due to a small number of channel complexes per hair cell (possibly 1-2 per stereocilium). There is ample evidence pointing to a relationship between the vertebrate hair cell and the cnidarian stereocilia-containing hair cells, which include nematocytes and other sensory ectodermal cells. We therefore asked if the nematocytes can be used as a model of the vertebrate hair cells for the characterization of the MET complex. We found that the architecture of the stereocilia that house the MET structures in the nematocytes of <em>Clytia hemisphaerica</em> polyps and medusa resemble, to some degree, those of vertebrate hair cells. Dependence on extracellular Ca2+, pharmacological susceptibility to ototoxic small molecules and the spontaneous incorporation of the fluorophore FM1-43 also support an evolutionary relation between nematocytes and vertebrate hair cells. We identified orthologs of vertebrate <em>TMC-1/2</em> genes in Clytiaand found that they are expressed in the nematocytes in the tentacles. To directly test the role of TMC-related ion channels in the cnidarian MET we used CRISPR/Cas9-mediated gene editing to mutate both <em>TMC </em>orthologs in Clytia. The resulting mutant lines will be evaluated by studying Clytia polyps stinging response and the ability of nematocytes to incorporate FM1-43 or a related dye.</p>
<hr /><p><strong><em>Phymactis papillosa</em></strong><strong>, a three-color pallet sea anemone in the Pacific Ocean</strong></p>
<p><span style="text-decoration: underline;">Yamaly Barrag&aacute;n</span>12, Pilar Haye3, Ben Titus4, and Estefan&iacute;a Rodr&iacute;guez2</p>
<p>1The City University of New York, New York, U.S.A</p>
<p>2American Museum of Natural History, New York, U.S.A</p>
<p>3Departamento de Biolog&iacute;a Marina, Universidad Catolica del Norte, Coquimbo, Chile</p>
<p>4The University of Alabama, Alabama, U.S.A</p>
<p>The sea anemone, <em>Phymactis papillosa, </em>is distributed along the eastern Pacific from Chile to Mexico. Within the geographical range, there are three biogeographic breaks representing distribution boundaries for several marine species: at 30S latitude in Chile (temperate water with frequent strong upwellings), at 24N latitude off El Salvador (sandy shorelines), and at 13N latitude in the south of the Gulf of California, resembling the sandy conditions of El Salvador coasts. This species presents three color variations (blue, green, and red). These morphotypes are not distributed evenly across the geographical range: all are found in the southern Pacific (cold and template water), whereas only the red color variety reaches the coast of Mexico (warm water). Based on collected samples, the three morphotypes cross the biogeographic break at 30S, but only the red one can cross the Tropical Eastern Pacific (TEP). The broad distribution across different biogeographic zones and the differential presence of the color morphotypes between biogeographic breaks make <em>P. papillosa</em> an interesting study case to test the origin of disjunct geographical distributions in sessile marine invertebrates: Did <em>P</em>. <em>papillosa</em> originate in Chile, where all morphotypes coinhabit, but only the red variety could cross the TEP region and colonize Mexico? To address this question, anatomical and genomics (RADseq) should be used to ensure all morphotypes belong to the same species. Whether they are cryptic species or not, migration rates, divergence, and gene flow between populations will be investigated to elucidate the genetic structure within widespread species.</p>
<p>Hasting, P.A. (2000). Biogeography of the Tropical Eastern Pacific: distribution and phylogeny of chaenopsid shed. Zoological Journal of the Linnean Society 319-335.</p>
<p style="text-align: justify;"><strong>Development of </strong><strong><em>NvFoxQ2d</em></strong><strong> expressing neural progenitor cells in </strong><strong><em>Nematostella vectensis</em></strong></p>
<p style="text-align: justify;">Natascha Bartsch1 and Fabian Rentzsch1</p>
<p style="text-align: justify;">1Department of Biological Sciences, University of Bergen, 5006 Bergen, Norway</p>
<p>The diversity of cell types in the nervous system is generated through populations of neural stem/progenitor cells which undergo several rounds of divisions during which their developmental potential becomes gradually restricted. The regulatory programs controlling the development from progenitor cells to differentiated nerve cells have been studied in a few organisms, but how these programs originated during evolution is not well understood. We are using the sea anemone <em>Nematostella vectensis</em> as a model to understand the evolution of nervous system development. In the background of a remarkable neurogenic potential and a comparably simple nerve net, we are studying the development of a progenitor cell population expressing the transcription factor <em>NvFoxQ2d</em>1. Interestingly, the <em>NvFoxQ2d</em> progenitor cells give rise to a population of spatially restricted sensory cells by one final division, allowing us to study the regulatory networks during this division in more detail. We generated a transcriptome of <em>NvFoxQ2d </em>expressing cells, which contained several enriched transcription factors that potentially regulate the development of this cell population. The temporal and spatial expression of some selected transcription factors was analyzed through <em>in-situ</em> hybridization (ISH) throughout early development and co-expression with <em>NvFoxQ2d</em> expressing cells was examined through double fluorescence ISH. Additionally, RNAi knockdown experiments were performed to analyze the functional relationship of the selected transcription factors. Overall, the project provides a more detailed description of the molecular regulation of nervous system development in an early-branching animal.</p>
<p>H. Busengdal, F. Rentzsch, Unipotent progenitors contribute to the generation of sensory cell types in the nervous system of the cnidarian Nematostella vectensis. Dev Biol 431, 59-68 (2017).</p>
<p style="text-align: justify;"><strong>Diving into sea anemone genomes</strong></p>
<p style="text-align: justify;">Charlotte Benedict1, Meg Daly1</p>
<p style="text-align: justify;">1The Ohio State University&nbsp;&nbsp;</p>
<p style="text-align: justify;">Whole genome duplication (WGD) events and proliferations of transposable elements (TE) in animal genomes have widespread evolutionary implications. WGD events have been previously regarded as an evolutionary landmark leading to the diversification of organisms, and rapid accumulation of TEs has been implicated in many cases of adaptive evolution. Previous research into anthozoan genomics has confirmed a WGD event in the scleractinian coral <em>Acropora digitifera</em>, however no studies have utilized available actiniarian genomes to investigate evidence of a previous WGD event in actiniarians. Additionally, the genome of <em>Exaiptasia pallida </em>revealed four classes of TEs with evidence for significant rapid expansion that aligns with the time of divergence with closely related anemone taxa.&nbsp;</p>
<p style="text-align: justify;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
<p style="text-align: justify;">In this study we 1) investigate the presence of whole genome duplication events across diverse sea anemone taxa using publicly available genomes and fourfold synonymous third-codon transversion rates (4DTv) and the synonymous substitution rates (dS) for orthologous genes and 2) compare the overall TE content and breakdown quantities of each type across diverse taxa.&nbsp;</p>
<p style="text-align: justify;">Guo et al. (2022) A myxozoan genome reveals mosaic evolution in a parasitic cnidarian. <em>BMC Biology </em>20:51&nbsp;</p>
<p style="text-align: justify;">Wilding et al. (2020) The genome of the sea anemone <em>Actinia equina</em> (L.): Meiotic toolkit genes and the question of sexual reproduction. <em>Marine genomics </em>53&nbsp;</p>
<p style="text-align: justify;">Voolstra et al. Comparative analysis of the genomes of <em>Stylophora pistillata </em>and <em>Acropora digitifera </em>provides evidence for extensive differences between species of corals. <em>Nature: Scientific Reports </em>7:17583&nbsp;</p>
<hr /><p><strong>Thermal stress promotes</strong><strong>pedal lacerate development in </strong><strong><em>Exaiptasia diaphana</em></strong></p>
<p><span style="text-decoration: underline;">Jun B. Cai</span>1, Kali Sivula1, Riley Jones1, Kathryn Hampton-Wonder1, 2022 OSU Invertebrate Biology Students1, Nate Kirk1, Virginia Weis1</p>
<p>1Oregon State University</p>
<p>The success of coral reefs depends on the symbiosis between corals and their dinoflagellate symbionts. Under heat stress, the dynamic homeostasis between host and algae destabilizes, leading to dysbiosis or bleaching where algae are lost from coral tissue. Bleached corals often die, leading to dramatic reef degradation. Climate change threaten to cause as much as 90% decline in reef cover by the end of the century. Given this threat, a better understanding of coral thermal tolerance is needed to develop solutions to the coral crisis. This includes quantifying the role of factors such as increasing temperature and differential nutrition in the development and growth of juvenile polyps. Using the sea anemone <em>Exaiptasia diaphana </em>(Aiptasia), a model system for the study of coral symbiosis, we studied the effects of temperature, nutrients and symbiosis on the development and survival of asexual buds, known as pedal lacerates. We examined the effects of symbiosis (symbiotic vs aposymbiotic), temperature (heat stress vs control) and nutrition (starved, normal feeding, enriched feeding) on the growth and development of tentacles in pedal lacerates. Our results show a significant impact of heat stress on the number of tentacles grown by pedal lacerates and no impact as a function of nutritional state.</p>
<p>Hoegh-Guldberg O, et al. 2017. Coral Reef Ecosystems under Climate Change and Ocean</p>
<p>Acidification. <em>Front. Mar. Sci</em>., 4.</p>
<p>Kipson S, et al. 2012. Effects of thermal stress on early developmental stages of a</p>
<p>gorgonian coral. <em>Mar Ecol Prog Ser</em> 470:69-78.</p>
<p>Logan CA, Dunne JP, Ryan JS. et al. 2021. Quantifying global potential for coral</p>
<p>evolutionary response to climate change. <em>Nat. Clim. Chang</em>. 11, 537&ndash;542.</p>
<p>Presnell JS, Wirsching E, Weis WM. 2022. Tentacle patterning during <em>Exaiptasia diaphana</em></p>
<p>pedal lacerate development differs between symbiotic and aposymbiotic animals.&nbsp; <em>PeerJ</em>, 10, e12770.</p>
<hr /><p><strong>Characterizing foot regeneration defects in </strong><strong><em>Hydra oligactis</em></strong></p>
<p>Craig Ciampa, Sergio E. Campos, and Celina Juliano</p>
<p>University of California, Davis</p>
<p>The Cnidarian <em>Hydra vulgaris</em> possesses a pool of constantly renewing stem cells that allow it to fully regenerate from any number of injuries. For this reason, <em>H. vulgaris</em> is of significant interest to researchers that aim to understand the molecular mechanisms of whole-body regeneration. Comparative approaches between closely related species with dramatically different regenerative abilities are powerful tools for uncovering key mechanisms underlying the regenerative ability. In a study by Hoffmeister et al. 1997, the species <em>Hydra</em><em>oligactis</em> was reported to be deficient in foot regeneration when bisected midway between the hypostome and foot. We have corroborated this observation and therefore we reason that this difference between <em>H. vulgaris</em> and <em>H. oligactis</em> could be leveraged to better understand the molecular mechanisms of regeneration. We aim to further characterize the regenerative ability of <em>H. oligactis</em> in order to test two hypotheses: 1) The reduced potential for full regeneration following foot amputation is correlated with the location of the wound along the oral-aboral axis, and 2) The deficiency in foot regeneration is caused by the failure of foot-specific transcription factor expression in response to injury. To test these hypotheses, we are characterizing the rate of foot regeneration in <em>Hydra oligactis</em> as compared to <em>H. vulgaris</em> after amputation at different locations along the oral-aboral axis. In addition, we are using our transcriptomic data to identify foot-specific genes and test their spatial expression over the course of foot regeneration using in situ hybridization.</p>
<hr /><p><strong>Insights into the role of symbiosis in NF-&kappa;B and innate immune function of </strong><strong><em>Exaiptasia pallida </em></strong><strong>&nbsp;following heat and nutrient exposure</strong></p>
<p>JK Da-Anoy1, Kian Thompson1, Maria Valadez Ingersoll1, Thomas D. Gilmore1, Sarah W. Davies1</p>
<p>1Department of Biology, Boston University, Boston, Massachusetts, 02215, USA</p>
<p>The Nuclear Factor-kappaB (NF-&kappa;B) is an evolutionarily conserved class of transcription factors throughout the animal kingdom that plays crucial roles in regulating physiological and immune-related processes. Recent work on cnidarian immunity, including NF-&kappa;B, showed dysregulation during coral bleaching, and increased NF-&kappa;B expression is correlated with increased thermotolerance in corals. Moreover, changes in expression of NF-&kappa;B were potentially linked to cnidarian symbiotic status, yet whether and how NF-&kappa;B and symbiotic state interact under increased temperatures and nutrient enrichment remain uncharacterized. To further understand these processes, we characterize the response of aposymbiotic and symbiotic Aiptasia following thermal (ramped to 35&deg;C: temperature increased by 2 &deg;C increments from 24 to 30 &deg;C, then by 1 &deg;C increment from 30 to 35 &deg;C), nutrient stress (nitrate-enrichment: 5 &mu;M), and their combination relative to ambient, low nutrient conditions. We compared NF-&kappa;B protein levels and innate immunity enzyme activity (phenoloxidase, catalase, peroxidase, total phenoloxidase) between aposymbiotic and symbiotic aiptasia across these treatments and characterized phenotypic changes and photosynthetic efficiency (Fv/Fm) to determine holobiont health. Future work will involve microbial (16S), gene expression, and proteomic profiling to determine the molecular signatures of these responses. Our findings will provide insights into how eutrophication stress can interact with thermal challenges to impact innate immunity, symbiosis, and ultimately host survival. Further elucidation of the molecular mechanisms underlying these processes may help understand how global change will affect immunity and shape future coral symbioses.</p>
<hr /><p><strong><em>Nematostella vectensis</em></strong><strong> as model organism to study species-specific physiological function of the human apoptosis channel TRPM2</strong></p>
<p><span style="text-decoration: underline;">Wiebke Ehrlich</span>1, James M. Gahan2, Fabian Rentzsch2 and Frank J.P. K&uuml;hn1</p>
<p>1University Hospital Aachen, Aachen, Germany</p>
<p>2Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway</p>
<p>In human cells the cation channel TRPM2 plays a major role in oxidative stress-mediated apoptosis and is also involved in regulation of core body temperature. Extensive studies on species variants revealed the complex activation mechanism of TRPM2. In particular, the functional characterization of the TRPM2 orthologue of <em>N. vectensis</em> allowed completely new insights into the gating mechanism of TRPM2, which apparently underwent fundamental changes during evolution. In our study we aimed to test whether these evolutionary differences also apply to the physiological function of TRPM2. For this purpose, we generated a loss-of-function mutation of TRPM2 in <em>N. vectensis</em> and performed a comparative analysis of the phenotypes of wildtype and mutant animals, especially after exposure to oxidative stress or high temperature. We could show that under standard conditions mutant and wildtype animals are indistinguishable in terms of morphology and development. However, exposure of the two experimental groups to the different stressors revealed that TRPM2 causes sensitization to oxidative stress but attenuates high-temperature injury. Thus, NvTRPM2 plays opposite roles in the cellular responses to oxidative stress and high temperature. Future studies on the molecular level should provide new insights into the physiological importance of TRPM2 in <em>N. vectensis</em> and may also deliver valuable information about the functional evolution of this important ion channel.</p>
<p>The study was supported by the Deutsche Forschungsgemeinschaft (DFG, grant KU 2271/4-3 to F.J.P.K.).</p>
<p>Ehrlich W, Gahan JM, Rentzsch F, K&uuml;hn FJP. TRPM2 causes sensitization to oxidative stress but attenuates high-temperature injury in the sea anemone Nematostella vectensis. J Exp Biol. 2022 Mar 15;225(6):jeb243717.</p>
<hr /><p><strong>Development of Genetically Encoded Voltage Indicator (GEVI) transgenic Hydra to study functional neural circuits with high spatiotemporal resolution</strong></p>
<p>Alondra Escobar1, Dhiraj Jain1, Kelly Kim1, Guillaume Duret1,&nbsp; J. T. Robinson1,2;</p>
<p>1Rice University, Houston, Texas</p>
<p>2Baylor College of Medicine, Houston, Texas</p>
<p><span style="color: #333333;">Understanding how a brain behaves is a long term goal in neuroscience. The study of functional neural circuits in a whole animal with high spatiotemporal resolution can help elucidate the millisecond timescale communication process that is involved in the recruitment and coordination of numerous cells to produce an output behavior. Therefore, the ability to record a large scale of neuron signals within a few milliseconds is key to understanding neural circuit behavior. Hydra Vulgaris is a great model to perform whole animal imaging since it has a small and simple nervous system that is well defined by 11 neuronal cell subtypes and 5 neural circuits. The gold standard electrophysiology patch clamp technique to measure neural activity is not possible in Hydra due to incompatible biophysical properties of the cell membrane. On the other side, a widely adopted optical technique to measure neural activity in Hydra are Genetically Encoded Calcium Indicators (GECIs). However, GECIs cannot resolve neural activity within a 5 millisecond time scale since they provide an indirect response of neural activity. Therefore, there is a need for tools that can measure neural activity in Hydra within millisecond timescale. Genetically Encoded Voltage Indicators (GEVIs) allow for optical recording of direct voltage changes across the cellular membrane, overcoming the temporal&nbsp; limitations of GECIs while maintaining large scale recording and cellular resolution. However, they have yet to be expressed in Hydra. In this study we evaluate, select multiple GEVIs and create transgenic animals to detect and measure for the first time voltage signals in Hydra Vulgaris. Further generation of GEVI transgenic Hydra lines can benefit the study of neural circuit reformation to regain function after an injury.</span></p>
<hr /><p><strong>Fluid Dynamics of a Nematocyst Firing and Ejection</strong></p>
<p><span style="text-decoration: underline;">Addie Harrison</span>, Laura Miller1, Christina Hamlet1, Wanda Strychalski1, Nick Battistia2</p>
<p>University of Arizona, Tucson, Arizona</p>
<p>1University of Arizona, Tucson, Arizona</p>
<p>1Bucknell University, Lewisburg, Pennsylvania</p>
<p>1Case Western Reserve University, Cleveland, Ohio</p>
<p>2The College of New Jersey, Ewing Township, New Jersey</p>
<p>Nematocysts are stinging organelles found in organisms such as jellyfish, anemones, and hydrozoans. In this study, we use numerical simulations to reveal the fluid dynamics of an idealized model of the barb accelerating towards the prey. Our preliminary results show that the nematocysts ability to accurately hit and puncture a surface it is projected to hit can vary based on the acceleration used to fire as well as the target's size and makeup</p>
<p>&nbsp;&nbsp;</p>
<p>Hamlet, C., Strychalski, W., &amp; Miller, L. (2020). Fluid dynamics of ballistic strategies in&nbsp;&nbsp;</p>
<p>nematocyst firing. Fluids, 5(1), 20.</p>
<p>Battista, N. A., Strickland, W. C., &amp; Miller, L. A. (2017). IB2d: a Python and MATLAB</p>
<p>implementation of the immersed boundary method. Bioinspiration &amp; biomimetics, 12(3), 036003.</p>
<p>NIH Grant GM132008</p>
<hr /><p><strong>An Investigation on the Evolution of Vision Related Genes in Box Jellyfish</strong></p>
<p><span style="text-decoration: underline;">Stuart Jaeger</span>1, Aide Macias Mu&ntilde;oz1, Todd Oakley1</p>
<p>1UC Santa Barbara</p>
<p>The evolution of vision has been of importance for researchers interested in the origin of complex traits due to the variation, complexity, and multiple origins of light-sensing structures. The phylum Cnidaria has emerged recently as an interesting group to study the genetic basis of complex trait evolution because there are multiple independent origins of eyes (at least 8) within the phylum. Among cnidarians, the most intricate eyes belong to cubozoans, which have twenty four eyes in total on four stalked, sensory structures called rhopalia. Rhopalia are a part of the central nervous system of the box jellyfish, and contain neurons and gravity sensors as well six eyes apiece, two of which are lensed eyes morphologically similar to vertebrate eyes. While our lab has previously conducted investigations into vision-related genes across independent eye origins, there has not been an investigation into the evolutionary relationships of vision-related genes within cubozoans. Here, we investigated expression of vision-related genes in three different species of Cubozoa, and built and examined phylogenetic relationships of these genes using transcriptome data from various tissues, including rhopalia. Because Cubozoans share an origin of eyes, we expected to find that across taxa, the same genes will be highly expressed in the eyes, and that the evolution of vision related genes will be largely conserved among Cubozoa. However, our phylogeny revealed instances of gene duplications and loss of eye development genes. A cubozoan-specific Rx gene was also found. This study reveals the diversity of gene expression and evolutionary history of vision genes within cubozoa.</p>
<p>Speiser DI <em>et al</em>. Using phylogenetically-informed annotation (PIA) to search for light-interacting genes in transcriptomes from non-model organisms. BMC Bioinformatics. 2014 Nov 19;15(1):350. doi: 10.1186/s12859-014-0350-x</p>
<p>Macias-Mu&ntilde;oz A <em>et al</em>. Evolution of Phototransduction Genes in Lepidoptera. Genome Biol Evol. 2019 Aug 1;11(8):2107-2124. doi: 10.1093/gbe/evz150</p>
<hr /><p><strong>Development of a genetic toolbox to study functional neural circuit behavior in </strong><strong><em>Hydra vulgaris</em></strong></p>
<p>Dhiraj M Jain1, Alondra Escobar2 &amp; Jacob T Robinson1,2,3,4</p>
<p>1Systems, Synthetic and Physical Biology Program, Rice University</p>
<p>2Department of Bioengineering, Rice University</p>
<p>3Department of Electrical and Computer Engineering, Rice University</p>
<p>4Department of Neuroscience, Baylor College of Medicine</p>
<p><em>Hydra</em>, a freshwater invertebrate first described in 1702,1 has been an organism of interest for the past 300 years. It is a promising model organism for neuroscience because of its simple nerve net, limited number of neuronal subtypes and the ability to regenerate functional neural circuits after damage. The ability to precisely monitor and manipulate neural circuits is essential to understanding the complex functions of the brain. Advancements over the last decade in optical techniques like voltage indicators and optogenetics have enabled neuroscientists to gain insight into neuronal activity by systematically manipulating neural circuits. Genetic manipulation in <em>Hydra </em>is carried out by creating transgenics using embryo microinjections. This process is time consuming with very low efficacy. Thus, development of a method to test fast transient expression of plasmid is an important need. This study aims to use electroporation of whole plasmid and mRNA can serve as an efficient tool for testing transient expression of a gene to ensure increased efficacy of microinjections.</p>
<p style="text-align: justify;">1. <span style="color: #222222;">van Leeuwenhoek, A. in </span><span style="color: #222222;"><em>The Collected Letters of Antoni van Leeuwenhoek</em></span><span style="color: #222222;"> Vol. XIV (ed. Palm, L. C.) 169&ndash;173 (Swets and Zeitlinger, 1996)</span></p>
<hr /><p><strong>Injury response: constructing a gene regulatory network in </strong><strong><em>Hydra vulgaris</em></strong></p>
<p>Iris Y Juanico, Jack F Cazet, and Celina E Juliano</p>
<p>University of California, Davis</p>
<p>Injury in animals triggers a conserved injury response, such as the activation of reactive oxygen species signaling, ERK signaling, and immune signaling pathways. Despite the conservation of the injury response, the subsequent regenerative response can vary widely depending on the species, life stage, and/or context of the injury. For example, the small freshwater cnidarian <em>Hydra </em>can regenerate its whole body from a small piece of tissue, whereas most mammals only have limited regenerative ability. To gain insight into how the injury response triggers regeneration in <em>Hydra, </em>our goal is to uncover the gene regulatory network (GRN) that connects the injury response to the activation of Wnt signaling, which is necessary and sufficient for head regeneration. MAPKs, specifically ERK, JNK, and p38, are phosphorylated upon injury, and their inhibition causes a block or defect in <em>Hydra </em>regeneration. In addition, four bZIP transcription factors (TFs), Jun, Fos, CREB, and Cr3l, are transcriptionally induced by injury. The putative TF binding motifs of bZIP TFs are found in the regulatory regions of <em>wnt3, wnt9/10c, </em>and <em>wntless</em>, and these motifs increase in accessibility in response to injury. Based on these findings, we propose the following model for the activation of Wnt signaling by injury: MAPK activity leads to the phosphorylation of bZIP TFs, thus activating these TFs, which in turn bind to the regulatory regions of <em>wnt3, wnt9/10c,</em> and <em>wntless </em>to induce transcription. We are currently testing our model using siRNA knockdowns of the bZIP TFs and MAPK inhibition. Then, we will analyze changes in the transcriptome and chromatin accessibility to deduce the GRN controlled by bZIP TFs and MAPKs during injury and regeneration.</p>
<hr /><p><strong>Can cnidarians learn? A novel neuroscience approach to rooting the evolution of memory.</strong></p>
<p>Hannah S. Justin, Will Alexander, Adam M. Reitzel</p>
<p>UNC Charlotte, Charlotte, North Carolina</p>
<p>The toolkit for studying neurobiology has expanded tremendously with advances in behavior, imaging, and sequencing, resulting in novel &lsquo;windows&rsquo; to characterize neural evolution and development. Current learning and memory research is predominantly conducted on bilaterian species with a CNS, with the vast majority in vertebrates.</p>
<p>An organism&rsquo;s ability to learn the environment can lead to adaptive behaviors with impacts on evolutionary fitness. To ascertain the evolutionary history of learning, it is important to assess the learning and variation in nervous system organization and function in species from groups that diverged prior to the bilaterian ancestor. This gap in the literature can be addressed by developing a new experimental system for neuroscience research: the sea anemone <em>Nematostella vectensis</em>. <em>Nematostella</em> has a dynamic behavior repertoire, and multiple separate subtypes of differentiated nerve cells, which cluster at the pharynx region of the animal1,2. Here, we report our early behavior experiments to examine the learning potential of <em>N. vectensis</em> in response to physical stimuli. These experiments will lead to future approaches using molecular and genomic approaches. <em>N. vectensis</em> provides a unique opportunity to examine how early learning mechanisms evolved and relate to the organization of a nervous system at a cellular level.</p>
<p>1. Leach, W. B, et. al, Transcriptome-wide analysis of differential gene expression in response to light:dark cycles in a model cnidarian. Comp Biochem Physiol Part D Genomics Proteomics 2018, 26, 40-49. 9.</p>
<p>2. Marlow, H. Q, et. al, Anatomy and development of the nervous system of Nematostella vectensis, an anthozoan cnidarian. Dev Neurobiol 2009, 69 (4), 235-54.</p>
<p><span style="color: #434343;"><strong>Stella-EP: </strong></span><span style="color: #434343;"><strong><em>Nemato</em></strong></span><span style="color: #434343; text-decoration: underline;"><strong><em>stella</em></strong></span><span style="color: #434343;"><strong><em> vectensis</em></strong></span><span style="color: #434343; text-decoration: underline;"><strong>E</strong></span><span style="color: #434343;"><strong>xpression and </strong></span><span style="color: #434343; text-decoration: underline;"><strong>P</strong></span><span style="color: #434343;"><strong>henotype Knowledgeable</strong></span></p>
<p><span style="color: #434343; text-decoration: underline;">Whitney Leach</span><span style="color: #434343;">1</span><span style="color: #434343;">, Lacey Ellington</span><span style="color: #434343;">1</span><span style="color: #434343;">, Stephanie Nowotarski</span><span style="color: #434343;">1</span><span style="color: #434343;">, Sofia Robb</span><span style="color: #434343;">1</span><span style="color: #434343;">, Matt Gibson</span><span style="color: #434343;">1</span></p>
<p><span style="color: #434343;">1</span><span style="color: #434343;">Stowers Institute for Medical Research</span></p>
<p><span style="color: #434343;"><em>Nematostella vectensis</em></span><span style="color: #434343;"> (Nvec) is a newly emerged model organism often used in comparative development, evolution of bilateral body plans, and regeneration-related studies. A plethora of valuable expression and phenotype data now exists for Nvec, accessible across a myriad of sources, technologies, and experimental paradigms: however, this information is unorganized and unmanaged, creating unnecessary challenges for researchers using these disparate data. Because Nvec is at a pivotal point where available published data is still amenable to manual curation, we have begun constructing Nvec anatomy and phenotype ontologies &ndash; including developmental stages, both of which will be used to underpin the curation of gene and phenotype data from current literature to ultimately create the </span><span style="color: #434343;"><em>Nemato</em></span><span style="color: #434343; text-decoration: underline;"><em>stella</em></span><span style="color: #434343;"><em> vectensis</em></span><span style="color: #434343; text-decoration: underline;">E</span><span style="color: #434343;">xpression and </span><span style="color: #434343; text-decoration: underline;">P</span><span style="color: #434343;">henotype Knowledgeable (Stella-EP). The curated data in Stella-EP will be interconnected within and across species using standardized vocabulary allowing diverse data to be associated and reviewable in ways that were previously inefficient or even impossible. Stella-EP will enable complex searches, such as returning all genes expressed in any part of the nervous system and any genes that are involved in phenotypes affecting the nervous system with corresponding developmental anomalies. While the scope of this work is focused on developing a knowledgebase for Nvec, the benefit of such resources invariably extends beyond Nvec: integration into multispecies resources, and generation of publicly available workflows using existing open-source software directly increases the impact of past, present and future publicly funded research.</span><span style="color: #434343;"> Because Stella-EP will be interoperable across species boundaries, it will further inform the biomedical community of gene functions and phenotypes associated with diseases spanning the tree of life</span><span style="color: #434343;">.</span></p>
<hr /><p><strong>Extracellular ATP is involved in epithelial wound healing in </strong><strong><em>Clytia hemisphaerica.</em></strong></p>
<p><span style="text-decoration: underline;">Elizabeth EL Lee</span>1, Isabel O&rsquo;Malley-Krohn2, Jocelyn Malamy1</p>
<p>1Department of Molecular Genetics and Cell Biology, The University of Chicago</p>
<p>2Biological Sciences Collegiate Division, The University of Chicago</p>
<p>Epithelial wound healing is a necessary process that allows organisms to repair integral barriers. While complex transcriptional responses are known to play important roles in wound healing, the earliest wound responses are transcriptionally-independent. One proposed transcriptionally-independent signal in wound healing is extracellular ATP (eATP) released by damaged cells. eATP binds to purinergic receptors and in some systems has been shown to promote cell migration and wound closure. However, the role of eATP in epithelial wound healing is poorly understood and difficult to study <em>in vivo</em>. We set out to characterize the influence of eATP in epithelial wound healing in the model system <em>Clytia hemisphaerica, </em>where healing is rapid and can be visualized at high resolution.&nbsp; We first tested a new technique, mesogleal microinjection (MGM), to introduce dyes and drugs to Clytia; previously, delivery of drugs and dyes to cnidarian tissues was a technical hurdle to experimentation. We found that molecules introduced via MGM diffuse readily throughout the mesoglea and reach the basal side of epithelial cells, while membrane permeable molecules enter epithelial cells from the mesoglea. We then used MGM to treat epithelial cells with actinomysin D to define transcriptionally-independent healing processes and found that wound closure occurs normally when transcription is inhibited, suggesting that healing involves transcriptionally-independent signaling pathways. We next tested agonists and antagonists of eATP signaling to assess the role of eATP in healing. We found that inhibition of eATP signaling reduced the rate wound closure. These results demonstrate a role for eATP signaling in epithelial wound healing in an <em>in vivo</em> system.&nbsp;</p>
<p>EL is supported by NSF PRFB 2011010</p>
<hr /><p><strong>A pipeline to rapidly validate commercial antibodies for cross-reactivity in </strong><strong><em>Clytia hemisphaerica</em></strong><strong>.</strong></p>
<p>Emily Watto, Zeeshan Banday, Elizabeth Lee and <span style="text-decoration: underline;">Jocelyn Malamy</span></p>
<p>Molecular Genetics and Cell Biology, The University of Chicago</p>
<p>Antibodies are excellent tools for localization of proteins.&nbsp; A set of validated antibodies that recognize key cellular components in a cnidarian model would be extremely useful.&nbsp; However, the vast majority of commercially available antibodies are raised against proteins from vertebrates and classical model systems, and cross reactivity to Cnidarian orthologous proteins is rarely (if ever) tested.&nbsp; Developing custom antibodies to cnidarian proteins is time consuming and costly.&nbsp; To address this bottleneck, we developed a pipeline to rapidly assess the cross-reactivity and specificity of commercial antibodies in the cnidarian model <em>Clytia hemisphaerica</em>.&nbsp; The approach is to use planula as a "factory" to express the Clytia protein of interest, and then validate a candidate antibody based on its ability to detect the expressed protein using Western blots and immunostaining.&nbsp; <em>Clytia</em> cDNA corresponding to the protein of interest is first amplified with primers that add a T7 promoter and a 3' 6x-His tag.&nbsp; RNA encoding the protein of interest is then transcribed<em> in vitro</em> from the <em>Clytia</em> cDNA amplicon, a polyA tail is added, and the RNA is microinjected into <em>Clytia</em> eggs.&nbsp; Fertilized <em>Clytia</em> eggs develop into planula within a day.&nbsp; We found that injected RNA is robustly expressed in planula, yielding sufficient protein for detection by Western blot.&nbsp; Progress in the use of this system to validate commercial antibodies to important <em>Clytia</em> proteins will be reported.</p>
<p>EL is funded by an ROL grant from the National Science Foundation</p>
<hr /><p><strong>Optimizing transgenesis by investigating egg production in </strong><strong><em>Hydra vulgaris</em></strong></p>
<p>Kamille Maningding*, Dinh Hieu Pham*, Pranathi Puttha*, Craig Ciampa, Malgosia Bzymek, and Celina Juliano</p>
<p>*These authors contributed equally</p>
<p>University of California, Davis</p>
<p>Transgenesis is a technique in which exogenous genes are incorporated into the genome of an organism. In the case of <em>Hydra vulgaris</em>, this process is achieved by injection of a fertilized egg with plasmids that carry fluorescent marker genes to allow for visualization of the transgenic tissues. When a <em>Hydra</em> embryo is injected with plasmid DNA, successful transgenesis occurs in between 10 to 20% of hatchlings. Although this is a relatively high rate of transgenesis, the development of new strains is slowed down by the low rate of egg production in <em>Hydra</em>. The aim of the work presented here is to maximize <em>Hydra</em> egg production and optimize injection time as essential steps towards increasing the number of transgenic lines that we can produce. To achieve this goal, we first expanded the populations of AEP <em>Hydra</em> strains available in the Juliano lab and recorded their population growth over time. These include the original AEP <em>Hydra vulgaris</em> strain and three F1 populations of this strain (referred to here as &ldquo; Mishima&rdquo;, &ldquo;Kiel&rdquo;, and &ldquo;Kiel Clone 2&rdquo;). The <em>Hydra</em> went through a period of clonal growth with frequent feeding to achieve critical population density. During this period, we determined the clonal growth rate for each strain. We found differences in growth rates between different populations of <em>Hydra</em>, primarily between the Mishima strain, which grew fastest within the experimental time frame, and the three other strains. We next decreased the density of the <em>Hydra</em> populations and decreased the feeding frequency to trigger egg production. The fastest growing strain (Mishima) did not produce eggs, while the slower growing strains produced between 3 and 10 eggs. Future directions include investigating if slowing down growth will lead to increased egg production in Mishima. Once we had eggs, our first goal was to measure the timing of the first cleavage stage of the embryo in order to optimize injection time for the most successful transgenesis rate. Our preliminary results suggest that on average Mishima embryos enter first cleavage 3 hours post fertilization, while the AEP and Kiel strains take 3.25 hours to enter that stage. We conclude that the optimal injection time is shortly before 3 hours post-fertilization. We plan to perform injections at various time points before first cleavage to determine if the timing of the injection influences the rate of transgenesis.</p>
<hr /><p><strong>Environmental Induction of a Novel Cell Type in the Scyphozoan Jellyfish </strong><strong><em>Cassiopea</em></strong></p>
<p>Cody Miner and Mark Q. Martindale</p>
<p>Whitney Laboratory for Marine Bioscience, University of Florida, St. Augustine, FL</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Also known as the upside-down jellyfish, <em>Cassiopea</em> is a genus of epibenthic Scyphozoan jellyfish found in shallow subtropical and tropical waters. <em>Cassiopea </em>forms an endosymbiotic relationship with <em>Symbiodiniaceae</em> dinoflagellate algae, as in zooxanthellate corals. It is unique in that after phagocytosis of algae, the host gastrodermal cells are induced to undergo an epithelial to mesenchymal transdifferentiating into &ldquo;amoebocytes,&rdquo; migrating into the mesoglea. This, in addition to the requirement of endosymbionts for strobilation, make <em>Cassiopea</em> an ideal model to study the interplay between symbiosis and development.</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
<p>We are initiating a project using modern molecular techniques to 1) describe transcriptional changes that occur during symbiotic establishment and amoebocyte induction using RNA-seq on specific cell types, 2) describe the key events required for amoebocyte transdifferentiation using functional disruption, and 3) to use genetic engineering to create a photosymbiotic host from the non-symbiotic anemone <em>Nematostella vectensis</em>. To these ends, we have developed a cell dissociation protocol to separate the mesoglea, gastrodermis, and epidermis, which, when combined with FACS sorting based on algal autofluorescence, allows for isolation of different symbiotic cell types within the same animal for bulk or single cell sequencing. This will be one of few studies to obtain cell type-specific transcriptomic data at multiple timepoints during symbiotic establishment, and the only to do so with multiple symbiotic cell types.</p>
<p>Colley, N. J., &amp; Trench, R. K. (1985). Cellular events in the reestablishment of a symbiosis between a marine dinoflagellate and a coelenterate [Article]. <em>Cell Tissue Res.</em>,<em> 239</em>(1), 93-103. <a href="https://doi.org/10.1007/bf00214908" target="_blank" rel="noopener"><span style="color: #0563c1;">https://doi.org/10.1007/bf00214908</span></a></p>
<p>Newkirk, C. R., Frazer, T. K., &amp; Martindale, M. Q. (2018). Acquisition and proliferation of algal symbionts in bleached polyps of the upside-down jellyfish, <em>Cassiopea xamachana</em> [Article]. <em>J. Exp. Mar. Biol. Ecol.</em>,<em> 508</em>, 44-51. <span style="color: #0563c1; text-decoration: underline;">https://doi.org/10.1016/j.jembe.2018.08.010</span></p>
<hr /><p><strong>Solving cell-type specific differences of the Wnt-directed gene regulatory network</strong><strong><em>.</em></strong></p>
<p>Hannah E Morris Little, Jack F Cazet, and Celina E Juliano</p>
<p>University of California, Davis</p>
<p>A fundamental question of developmental biology is to understand how a limited number of signaling pathways direct the specification of many cell types. One such pathway is the canonical Wnt signaling pathway which is highly conserved across animals and plays a role in a myriad of developmental processes. To direct different developmental outcomes, Wnt signaling must activate different <span style="text-decoration: underline;">gene regulatory networks (GRNs)</span> in different contexts. To reveal general principles of how Wnt ligands can activate unique GRNs, I am using <em>Hydra vulgaris</em> to discover how two distinct cell types, the oral endoderm and ectoderm, uniquely respond to the Wnt signaling pathway. To activate target gene expression in specific developmental contexts, Bcat/TCF, the effector <span style="text-decoration: underline;">transcription factors (TFs)</span> of Wnt signaling, must work in a combinatorial fashion with other TFs, but the identify of these are largely unknown. The goal of this project is to identify TFs that facilitate the activation of Wnt targets in <em>Hydra</em>. To globally identify direct targets of Wnt signaling, I first identified Wnt-responsive targets by analyzing RNA-seq data collected from Alsterpaullone (Alp) treated animals and found ~1150 Alp-responsive genes. To determine which were direct targets, I examined the regulatory regions of the Alp-responsive genes for accessible TCF TF binding sites and identified 284 putative direct targets of Bcat/TCF in <em>Hydra</em>. I then defined the spatial expression patterns of these putative direct targets using our lab&rsquo;s single cell expression map. To identify putative combinatorial regulators active in the epithelial cells, I performed motif enrichment analysis on accessible regulatory regions of putative direct Bcat/TCF targets expressed in the oral epithelium and found that Homeobox TF binding motifs are enriched. These data support the hypothesis that Homeobox TFs act as cell-type specific combinatorial regulators of direct Wnt targets. In the future, I will further test this hypothesis via siRNA knockdown of Homeobox TFs in epithelial cell to test if they are required for oral cell specification as well as test the effect on direct Wnt targets. This will further our understanding of how a single signaling pathway is able to activate different downstream GRNs in different contexts.</p>
<hr /><p><strong>Decoding scaling mechanisms in </strong><strong><em>Hydra Vulgaris</em></strong></p>
<p><span style="text-decoration: underline;">Clara Nuninger</span><em>1,2</em>, Panagiotis Papasaikas<em>1,3</em>, Charisios D. Tsiairis<em>1</em></p>
<p style="text-align: justify;">1Friedrich Miescher Institute for Biomedical Research, Basel, Switzerland</p>
<p style="text-align: justify;">2University of Basel, Basel, Switzerland</p>
<p style="text-align: justify;">3SIB Swiss Institute of Bioinformatics, Basel, Switzerland</p>
<p style="text-align: justify;">Deciphering the mechanisms that regulate scaling of tissue patterns during development and regeneration is a longstanding challenge. The freshwater cnidarian <em>Hydra</em> is a powerful model for exploring scaling mechanisms. Hydra&rsquo;s body resembles a tube that is composed of epithelial, pluripotent cell populations and is organized along the main oral-aboral axis. At the two opposing poles of the animal axis, terminally differentiated cells form the tentacles and the foot of the animal1. Hydras are well known for their ability to continuously renew all of its cells while maintaining a constant size and adapt its size to environmental conditions e.g. food-supply and temperature2. This plasticity of tissue highlights a certain adjustment of the intrinsic scale of patterning to some measure of the system size. In order to understand how this axial pattern scale, we focus on gene expression pattern along the main axis of <em>Hydra</em>. A great number of genes have a graded expression profile along the main axis and this profile scales with the animal size. We exploit mutant animals with altered sizes to understand how these graded patterns adapt to different sizes through molecular and quantitative approaches. Taken together, this project will contribute to shed the light on how self-organizing pattern allows tissues to sense and adapt to size, in a model system that evolutionary lies at the origin of multicellular organization.</p>
<ol>
<li><span style="color: #1e1e1e;">Mart&iacute;nez DE, Bridge D. Hydra, the everlasting embryo, confronts aging. Int J Dev Biol. 2012; 56(6- 8):479-87.</span></li>
<li><span style="color: #1e1e1e;">Mortzfeld BM, Taubenheim J, Klimovich AV, Fraune S, Rosenstiel P, Bosch TCG. Temperature and insulin signaling regulate body size in Hydra by the Wnt and TGF-beta pathways. Nat Commun. 2019 Jul 22; 10(1):3257.</span></li>
</ol>
<hr /><p><strong>Anomalies in the mitochondrial genomes of members of Aliciidae, Boloceroididae, and Gonactiniidae (Actiniaria:Metridioidea)</strong></p>
<p><span style="text-decoration: underline;">Isabel Pen</span>1, Michael Broe1, and Marymegan Daly1</p>
<p>1The Ohio State University, Columbus, Ohio</p>
<p>Analysis of gene sequences from 3 mitochondrial and 2 nuclear genes for a broad sampling of actiniarian taxa proposed a novel grouping of the families Aliciidae, Boloceroididae, and Gonactiniidae. (Rodr&iacute;guez et al., 2014). Although previously unlinked, the association of these taxa was credible given the consistency of the genetic data and the shared occurrence of anatomical traits, including complex column structures (Alic., Bolo.), tentacular sphincters (Bolo., Gona.), and retention of juvenile musculature in adults (Bolo., Gona.). Although the mitogenome structure of actiniarians is generally conserved, complete mitogenomes identify unusual gene orders (<em>Alicia sansibarensis</em>: Foox et al. 2016) and genome structures (<em>Protanthea simplex</em>: Dubin et al. 2019) within this clade. The structure in <em>P. simplex</em> is especially unusual &ndash; a pair of circular genomes rather than a single circular genome. We have reconstructed a complete mitogenome for the aliciid <em>Lebrunia danae</em> from transcriptomic data to understand mitogenome diversity and evolution within this clade. Like its confamilial <em>A. sansibarensis</em>, <em>L. danae</em> has a mitochondrial gene order that deviates from typical actiniarians. This finding suggests that the genome of members of this clade are evolving in ways unlike those of other anemones, which adds nuance to the recent grouping of these lineages into a clade. These findings add to a growing picture of complexity in cnidarian mitogenomes.</p>
<p>Rodr&iacute;guez, E., et al. 2014. Hidden among sea anemones: the first comprehensive phylogenetic reconstruction of the order Actiniaria (Cnidaria, Anthozoa, Hexacorallia) reveals a novel group of hexacorals. PloS One, 9(5): p.e96998.</p>
<p>Dubin, A., et al. 2019. Deep-water sea anemone with a two-chromosome mitochondrial genome. Gene, 692:195-200.</p>
<p>Foox, J., et al. 2016. Multiplexed pyrosequencing of nine sea anemone (Cnidaria: Anthozoa: Hexacorallia: Actiniaria) mitochondrial genomes. Mitochondrial DNA Part A, 27: 2826-2832.</p>
<hr /><p><strong>Towards identifying the evolutionarily conserved mechanism of ring canal formation</strong></p>
<p><span style="text-decoration: underline;">Kari L. Price</span>, Dyuthi M. Tharakan, Kathleen Ayers, Vivian Taylor, Lynn Cooley</p>
<p>Yale University School of Medicine, New Haven, CT</p>
<p>Throughout the animal kingdom, the development of male and female gametes relies on the formation of intercellular bridges, called ring canals, which connect germ cells in a syncytium. A regulated program of incomplete cytokinesis, marked by the absence of abscission, ensures the production of ring canals. Comparison of the conserved cytokinesis and ring canal protein MKLP1/KIF23/Pav in Drosophila, mouse, and Hydra has revealed shared features of incomplete cytokinesis and suggests that an evolutionarily conserved mechanism underlies ring canal formation across biology. To gain new mechanistic insights, we are characterizing Hydra ring canals to identify additional shared components and features involved in ring canal biogenesis.</p>
<p>Using the antibody we made against KIF23 (Price et al., 2022), we discovered KIF23-labeled ring canals in many cells of the interstitial cell lineage, including both the germline and somatic cells. Using commercially available antibodies to known ring canal components in other systems, we found that Hydra ring canals are also enriched in phosphotyrosine epitopes and F-actin, like Drosophila ring canals but distinct from mouse ring canals. We are currently investigating the localizations of other candidate ring canal components co-enriched with KIF23 transcripts in single cell RNA sequencing data (Siebert et al., 2019), and isolating KIF23-interacting proteins via immunoprecipitation coupled with mass spectrometry. With FIB-SEM, we have generated tomographs of Hydra germline ring canals that have revealed new structural details.</p>
<p>Price, K.L., Tharakan, D.M., Cooley, L. (2022) Evolutionarily conserved midbody reorganization precedes ring canal formation during gametogenesis. bioRxiv. doi:10.1101/2022.06.03.494691</p>
<p>Siebert, S., et al. (2019) Stem cell differentiation trajectories in Hydra resolved at single-cell resolution. Science. doi:10.1126/science.aav9314</p>
<hr /><p><strong><em>De-novo </em></strong><strong>Transcriptome and Genome of the Edwardsiid Anemone </strong><strong><em>Edwardsia elegans</em></strong></p>
<p><span style="text-decoration: underline;">Auston Rutledge</span>1, Remi Ketchum1, Jason Macrander1,2, Edward Smith1, Adam Reitzel1</p>
<p>1University of North Carolina Charlotte, Charlotte, United States</p>
<p>2Florida Southern College, Lakeland, Florida, United States</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<em>Edwardsia elegans</em> is a burrowing anemone commonly found on the North Atlantic coast of North America. They are one of about 90 species in the family Edwardsiidae<em>, </em>which is an informative group for research into the relationships of actiniarians and evolution in the Anthozoa. This species is a confamial with the lined anemone <em>Edwardsiella lineata </em>and the commonly used model cnidarian <em>Nematostella vectensis</em>, but there remains little sequence information for species in the <em>Edwardsia</em> genus and no genomic/transcriptomic resources. Here we have generated a <em>de-novo </em>transcriptome and genome of the anemone <em>Edwardsia elegans</em>. The transcriptome was generated using RNA-seq data, resulting in approximately 24,000 genes of which about 75% are annotated with a characterized protein from the <em>Nematostella </em>transcriptome or NCBI and Uniprot databases. Pfam analysis shows a high similarity between our <em>de-novo</em><em>E. elegans </em>transcriptome and the available <em>Nematostella </em>transcriptome, with similar numbers of genes for transcription factor families and gene families involved in cellular defense pathways. This along with a high BUSCO score (95% complete) and an N50 of 2,435nt shows that our transcriptome is of comparable quality to other cnidarians. We also sequenced and annotated the genome of <em>Edwardsia elegans </em>(~330Mb)using a combination of Illumina and Oxford Nanopore. The <em>Edwardsia </em>genus is a paraphyletic group and our newly generated data may help shed light on the phylogeny of this group of actinarians and allow for improved comparative genomics, molecular, and organismal studies of both anthozoans and cnidarians as a whole.</p>
<p>Daly, M. (2002). A systematic revision of <em>Edwardsiidae</em> (Cnidaria, Anthozoa). <em>Invertebrate&nbsp;</em></p>
<p><em>Biology</em> 121.3.</p>
<p>Funding: SPRINT award from the University of North Carolina Charlotte</p>
<hr /><p><strong>Single-Cell Sequencing of Functionally Isolated Coral Cell Populations</strong></p>
<p>G.A. Snyder1, A.M. Bonacolta1, S. Eliachar2, S. Talice2, U. Hadad3, O. Gershoni-Yahalom2, W.E. Browne4, B. Rosental2 and N. Traylor-Knowles1</p>
<p>1Department of Marine Biology and Ecology, Rosenstiel School of Marine and Atmospheric Science, University of Miami, Miami, FL, USA</p>
<p>2The Shraga Segal Department of Microbiology, Immunology, and Genetics, Faculty of Health Sciences. Regenerative Medicine and Stem Cell Research Center, Ben Gurion University of the Negev, Beer Sheva, Israel</p>
<p>3Ilse Katz Institute for Nanoscale Science and Technology, Ben-Gurion University of the Negev, Beer Sheva, Israel</p>
<p>4Department of Biology, University of Miami, Coral Gables, FL, USA</p>
<p>Single-cell sequencing from whole-tissue samples has been accomplished in Cnidarians several times1-3. Here we build upon these methods by applying Drop-Seq to 19 cellular populations from the branching Indo-Pacific scleractinian species <em>Pocillopora damicornis</em>. These populations were isolated with fluorescence-activated cell sorting (FACS), utilizing various non-species-specific markers, which included assays for stem cells as well as putative immune cells4. Sequencing individual populations will help reduce bias against rare cell types, as well as pair cell populations identified through gene expression to functional and morphological data. Furthermore, this method will aid the description of poorly characterized cell types within corals, including the putative immune and stem cells. Genetically characterizing these cell populations will confirm not only functional specialty and morphology, but aid in downstream localization within the tissue, identification of different disease causes, and in the identification of cell-specific responses to stressors such as temperature and ocean acidification.&nbsp;</p>
<p>References:</p>
<p>1Seb&eacute;-Pedr&oacute;s et al., 2018, Cell <em>173</em>, 1520-1534&nbsp;</p>
<p>2Hu et al., 2020, Nature <em>582</em>, 534-538</p>
<p>3Levy et al., 2021, Cell <em>184</em>, 2802-2804</p>
<p>4Snyder et al. 2021, Front. Immunol. <em>12</em></p>
<hr /><p><strong>Characterizing Stem Cell Heterogeneity in </strong><strong><em>Hydractinia symbiolongicarpus</em></strong><strong> with Methanol-Fixed 10X Single-Cell RNA-sequencing</strong></p>
<p><span style="text-decoration: underline;">Jingwei Song</span>1,2, Andreas D. Baxevanis3, Christine E. Schnitzler1,2</p>
<p>1Whitney Laboratory for Marine Bioscience, University of Florida, St. Augustine, USA</p>
<p>2Department of Biology, University of Florida, Gainesville, USA</p>
<p>3National Human Genome Research Institute, National Institutes of Health, Bethesda, USA</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Studying stem cells from a wide range of taxa holds promise for ultimately developing stem-cell-based regenerative medicine therapies for humans. Recent studies in planarians and acoels reported adult stem cell heterogeneity based on single-cell transcriptomic profiles, suggesting a previously unappreciated complexity involved in this population of cells. <em>Hydractinia symbiologicarpus</em>, a colonial marine hydroid, has a population of pluripotent adult stem cells called i-cells. Similar to planarians and acoels, we hypothesized that the i-cells of <em>Hydractinia</em> are transcriptionally heterogeneous and could even be classified into subtypes. To test this hypothesis and the feasibility of using methanol-fixed <em>Hydractinia</em> cells for single-cell RNA-sequencing (scRNAseq), 20 polyps were mechanically dissociated and fixed with 80% methanol and the cells were processed with standard 10X scRNAseq protocols. The quality of the sequencing reads was comparable to a previous dataset using live cells. After integrating the two datasets with CCA, we identified all previously annotated cell types, including two clusters of i-cells. We discovered that mechanical dissociation resulted in high levels of ambient RNA in the resulting dataset whereas enzymatic dissociation did not, thus we recommend enzyme-based dissociation for <em>H. symbiolongicarpus</em>. Future research will include adding more replicates of fixed cells for scRNAseq to increase the number of i-cells to better characterize heterogeneity in this population.</p>
<p>Zeng, A., et al. (2018). Prospectively isolated tetraspanin+ neoblasts are adult pluripotent stem cells underlying planaria regeneration. <em>Cell</em>, <em>173</em>(7), 1593-1608.e20.</p>
<p>Hulett, RE., et al. (2022) Acoel single-cell atlas reveals expression dynamics and heterogeneity of a pluripotent stem cell population.bioRxiv 2022.02.10.479464</p>
<hr /><p><strong>Light Sensation and Regeneration in </strong><strong><em>Hydra vulgaris</em></strong></p>
<p><span style="text-decoration: underline;">Marina Stoilova</span>1, Aide Macias Munoz1,&nbsp; Natasha Picciani2,&nbsp; Todd Oakley1</p>
<p>1UC Santa Barbara</p>
<p>2Yale University</p>
<p>The ability to sense and respond to light is generally found in neuronal cell types (such as photoreceptors), but some non-neuronal cells are also photosensitive. Many of these instances of non-neuronal light sensitivity occur in Cnidarians, where visual systems range from basic to highly complex. Here, we focused on the freshwater hydrozoa <em>Hydra vulgaris</em>, which exhibits photosensitivity despite lacking visual structures. First, we examined the expression of light-sensing protein genes (Opsins) across neuronal and non-neuronal cell types. Using published single-cell RNA sequencing data, we identified 29 candidate opsin gene transcripts and plotted their expression across <em>Hydra</em> cell types and cell states<em>.</em> We found that some candidate opsin genes are expressed in consecutive non-neuronal cell states, potentially following the progression of cell differentiation. Upon further investigation, these candidate opsin genes appear to increase in expression in the <em>Hydra</em> head organizer<em>.</em> Some opsins also shared transcription factor binding sites associated with developmental genes for patterning and positional identity. This led us to ask what role light sensation may play in regeneration in <em>Hydra. </em>To investigate further, <em>Hydra</em> are bisected and allowed to regenerate in light and dark conditions. This is then repeated using the CNG pathway inhibiting drug cis-diltiazem, which disrupts the phototransduction cascade that allows opsins to communicate to the rest of the cell. The <em>Hydra</em> are induced to regenerate in light and dark conditions with and without exposure to cis-diltiazem. These two experiments will help to shed light on how a complex process such as regeneration can be mediated by an environmental stimulus such as light.</p>
<p>EEMB Worster Award</p>
<p>Siebert, S. <em>et al.</em> Stem cell differentiation trajectories in Hydra resolved at single-cell resolution. doi:10.5061/dryad.v5r6077<a href="http://paperpile.com/b/RGIXtq/gc7ey" target="_blank" rel="noopener">.</a></p>
<p>Plachetzki, D. C., Fong, C. R. &amp; Oakley, T. H. Cnidocyte discharge is regulated by light and opsin-mediated phototransduction. <em>BMC Biol.</em><strong>10</strong>, 17 (2012).</p>
<hr /><p><strong>Classifying the Cell-Specific Microbiome of the Coral, </strong><strong><em>Pocillopora damicornis</em></strong></p>
<p><span style="text-decoration: underline;">Aliyah True</span><span style="text-decoration: underline;">1</span>, Grace Snyder1, Rowan Thomas1, Nikki Traylor-Knowles1</p>
<p>1University of Miami, Rosenstiel School of Marine and Atmospheric Sciences, Miami, Florida</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The innate immune system is responsible for the ability to recognize self from non-self; an important facet of multicellular life. The cnidarian microbiome (which include archaea, bacteria, fungi, protists, and viruses) play versatile regulatory roles for the sake of holobiont health including reproduction, digestion, and immune response. Classifying the microbes of specific cell types can give insight into the symbiotic interactions that contribute to the distinct microbiomes of a compartmentalized organism. We used Fluorescence Activated Cell Sorting (FACS) and sequencing of 16S rRNA genes to classify the cell specific microbiome of <em>Pocillopora damicornis.</em> The goal of this experiment is to find distinctions between the environment, the metaorganism, and it&rsquo;s compartmental layers (the mucus layer and coral tissues). Our hypothesis is that specific cell populations will have a microbiome different from and more stable to their ever changing surroundings. The potential of multiple bacterial communities associated with specific cell types will pave the way for explorations of coral microbial symbiosis. This will inform the role microbial heterogeneity plays in an individual and the potential selective advantage each metazoan has in response to its environment. This is a necessary step in understanding how the innate immune system contributes to establishing and maintaining a distinct cell specific microbiota outside of dinoflagellate symbionts. In the future, this can help us to better understand microbial community fluctuations during disease outbreaks and potentially help in conservation and restorative efforts.</p>
<p>Dzik, J. M. (2010).<em> The ancestry and cumulative evolution of immune reactions</em>. Acta biochimica Polonica.</p>
<p>Esser, D., et al. (2018).<em> Functions of the microbiota for the physiology of Animal Metaorganisms</em>. Journal of Innate Immunity, 11(5), 393&ndash;404.</p>
<hr /><p><strong>Revealing adult stem cell heterogeneity in </strong><strong><em>Hydractinia</em></strong><strong> using a combination of single-cell transcriptomics and experimental molecular techniques</strong></p>
<p><span style="text-decoration: underline;">Justin Waletich</span><span style="text-decoration: underline;">1</span>,2, Amanda Yeo1,2, Danielle de Jong1,2, Christine E. Schnitzler1,2</p>
<p>1Whitney Laboratory for Marine Bioscience, University of Florida, USA</p>
<p>2Department of Biology, University of Florida, Gainesville, USA</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Adult stem cells are known to confer incredible abilities, such as unlimited regeneration and immortality, to animal taxa such as planarians and <em>Hydra</em>. The cnidarian <em>Hydractinia</em><em>symbiolongicarpus</em> has remarkable abilities for infinite growth, lack of aging, and unlimited regeneration, due to its population of adult stem cells, known as interstitial stem cells or &lsquo;i-cells&rsquo;.&nbsp; Our goal is to characterize the i-cell population in <em>Hydractinia</em> to better understand the vast potential these cells have in generating all cell types within the adult animal. <strong>It is unknown if all i-cells have the same potential to generate all cell types, or if there are subpopulations of i-cells with differing or more restricted capabilities in the animal</strong>. Using single-cell transcriptomics, we created a cell-type atlas for adult <em>Hydractinia</em> and found two separate cell clusters with known stem cell marker expression. One cluster was connected to a germ cell fate while the other was linked to a somatic fate. We obtained highly expressed marker genes from each cluster, as well as markers expressed in both, to generate a list of ~700 potential i-cell candidate genes. We are spatially characterizing the expression pattern of eight of these top i-cell markers, with some exclusive to one cluster and some present in both, using <em>in situ</em> hybridization. Results indicate heterogeneity in expression patterns among markers in different polyp types and life stages. We have also started to generate fluorescent transgenic reporter lines for a subset of the i-cell markers to further understand their contribution to different cell lineages and to regenerating tissues. This work will provide a foundation for understanding adult stem cell heterogeneity in <em>Hydractinia </em>and bring further insights into their incredible regenerative abilities.</p>
<hr /><p><strong>The role of parental partitioning of nutrients in </strong><strong><em>Exaiptasia diaphana </em></strong><strong>pedal lacerate development</strong></p>
<p><span style="text-decoration: underline;">Erick White</span>1, Lily Rice1, Deeann Schall1, Virginia Weis1</p>
<p>1Oregon State University</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The success of coral reefs in oligotrophic waters depends on the cnidarian-dinoflagellate symbiosis that forms via translocation of nutrients between partners. The propagation of many corals is partially achieved by reproduction of polyps including by budding and fission. However, very little is known about how energetics and nutrition factor into the partitioning of resources from the parent to the offspring for development. Here, we use the sea anemone <em>Exaiptasia diaphana </em>(Aiptasia), a model system for the study of coral-algal symbiosis because it hosts the same family of dinoflagellates as corals and can exist without symbionts, which allows to gain insight into the role of nutrition and symbiosis in budding, known as pedal laceration, and development in corals. Aposymbiotic and symbiotic parents were maintained in artificial sea water at 25&deg;C and fed brine shrimp three times a week; lacerates were then cut from the parents and harvested at different timepoints for analysis. To quantify lipid localization and abundance, we used a combination of imaging and spectrophotometry via the lysochrome dye Oil Red O which stains neutral lipids and triglycerides. Our results showed a significant difference in lipid abundance between symbiotic and aposymbiotic animals across all timepoints. Staining was concentrated around the pedal and oral disk in lacerates as well as along ostensible mesentery lines. Future experiments will explore the role of nutritional history in lacerate development and lipid abundance.</p>
<p>Douglas AE. 2010. The symbiotic habit. Princeton University Press.</p>
<p>Presnell JS, Wirsching E, Weis VM. 2022. Tentacle patterning during Exaiptasia diaphana pedal lacerate development differs between symbiotic and aposymbiotic animals. <em>PeerJ</em><strong>10</strong>:e12770</p>
<p>Yellowlees D, Rees TA V., Leggat W. 2008. Metabolic interactions between algal symbionts and invertebrate hosts. <em>Plant, Cell Environ</em><strong>31</strong>:679&ndash;694</p>
<p>This work was supported by an NSF award IOS:EDGE:1529059 to Virginia Weis.</p>
<hr /><p><strong>Characterizing the Hippo Pathway in </strong><strong><em>Hydractinia</em></strong><strong> Interstitial Stem Cells</strong></p>
<p><span style="text-decoration: underline;">Qingru Sue Xu</span>1,2, Christine E. Schnitzler1,2</p>
<p>1 Whitney Laboratory for Marine Bioscience, University of Florida, St. Augustine, USA.</p>
<p>2 Department of Biology, University of Florida, Gainesville, USA.</p>
<p>Stem cells are defined by their ability to indefinitely self-renew and their production of differentiating progeny. The marine, colonial invertebrate <em>Hydractinia</em>, like other members of the cnidarian class Hydrozoa, has an adult stem cell population called interstitial stem cells, or i-cells. The i-cell population in <em>Hydractinia</em> is considered pluripotent, because it can give rise to all cell lineages, including both somatic and germ cells. <em>Hydractinia</em> colonies maintain tissue homeostasis throughout their adult life by i-cell proliferation, migration, and differentiation, but the specific genetic mechanisms that control and maintain i-cell activity and behavior remain largely unknown. We propose that a potential i-cell regulator is the Hippo signaling pathway. This pathway has established functions in regulating stem cell proliferation and cell fate in other animals including mammals, <em>Drosophila</em>, and planarians. We are conducting spatial expression studies of key genes in the Hippo pathway using colorimetric <em>in situ</em> hybridization during different <em>Hydractinia</em> life stages and during adult polyp head regeneration. We are also combining fluorescent <em>in situ </em>hybridization with an EdU assay (marker of proliferating cells) and with immunofluorescence of the i-cell marker Piwi1, to find the association between Hippo pathway genes, i-cells and proliferating cells in general. Future work will include functional manipulation (knock down and/or knock out) of Hippo pathway genes to investigate the effect on i-cell proliferation and function during homeostasis and regeneration. This study underscores the role of the Hippo pathway in regulating pluripotent adult stem cells in an early-branching animal, furthering our understanding of adult stem cell function and evolution.</p>
<hr /><p><strong>Developing New Cell Type Marker Genes in the Colonial Hydroid </strong><strong><em>Hydractinia</em></strong></p>
<p><span style="text-decoration: underline;">Amanda Yeo</span>1,2, Danielle de Jong1,2, Justin Waletich1,2, Christine E. Schnitzler1,2</p>
<p>1Whitney Laboratory for Marine Bioscience, University of Florida, St. Augustine, USA</p>
<p>2 Department of Biology, University of Florida, Gainesville, USA</p>
<p>As the sister group to bilaterians, cnidarians hold an informative phylogenetic position for understanding the origin of cell types, fundamental gene functions, and how they have changed throughout evolution. The colonial hydroid <em>Hydractinia symbiolongicarpus</em> is capable of infinite growth and unlimited regeneration, while also exhibiting a lack of aging, making it an excellent cnidarian model system. Cnidarians possess a small number of known terminally differentiated somatic cell types including epithelial cells, gland cells, neurons, and cnidocytes. In hydrozoan cnidarians, a population of adult stem cells, known as interstitial stem cells (i-cells), gives rise to all of these somatic cell types as well as germ cells. Our lab is interested in developing reliable cell type marker genes for each of the cell types in <em>H</em>. <em>symbiolongicarpus</em>. The focus of this project is to develop epithelial and i-cell markers. This involves utilizing colorimetric and fluorescent <em>in situ </em>hybridization to conduct spatial expression studies of potential marker genes that appear to be specifically expressed in these cell types, based on our single-cell RNAseq atlas. Future work will also include exploring their expression in different contexts such as regeneration, and in different life stages. Once validated as cell-type-specific marker genes, we will proceed with the creation of fluorescent transgenic reporter animals to highlight these cell types, allowing us to study them in greater depth. This work will provide a foundation for understanding the origins of each of the different cell types in <em>Hydractinia</em>.</p>
<p>Gahan, J. M. et al. The interstitial stem cells in <em>Hydractinia</em> and their role in regeneration. Current Opinion in Genetics &amp; Development. 40, 65-73 (2016).</p>
<hr /><p><strong>Phylogenetic Analyses of Acuticulate Sea Anemones (Cnidaria, Actiniaria, Metridioidea) Resolve Taxonomic Uncertainties and Reveal New Insights on Relationships</strong></p>
<p>Mengjin Zhang, Alonso Delgado, Marymegan Daly, Heather Glon</p>
<p>The Ohio State University, Columbus, USA</p>
<p>Understanding biodiversity and evolutionary history among sea anemones have been hampered by the reliance on morphology-based taxonomies, given the many shared external traits between groups. Broad-scale phylogenetic analyses of genetic data have addressed intraordinal relationships, identifying major lineages that are stable across dataset and taxon sample. However, relationships among and within families remain problematic, with key nodes unresolved or resolved differently across analyses.</p>
<p>Acuticulata is among the stable major lineages identified within Actiniaria. This subclade of Metridioidea includes the model taxon <em>Exaiptasia pallida</em> and the invasive species <em>Diadumene lineata</em> and <em>Metridium senile</em>. Interpreting, generalizing, and predicting genomic and biological data for these important taxa is hindered by a lack of understanding of sister lineages or character transformations. In this study, we present a phylogenetic re-evaluation of the Acuticulata using a 5-gene makers dataset with denser taxonomic sampling. We include 106 samples and construct phylogenies using different methods. Our results identify strong candidates for the sister lineage to Aiptasiidae and suggest that the boundaries of Aiptasiidae need to be broadened.</p>
<p>Grajales, A., &amp; Rodr&iacute;guez, E. (2016). Elucidating the evolutionary relationships of the Aiptasiidae, a widespread cnidarian&ndash;dinoflagellate model system (Cnidaria: Anthozoa: Actiniaria: Metridioidea). Molecular Phylogenetics and Evolution, 94, 252-263.</p>
<p>Grajales, A., &amp; Rodriguez, E. (2014). Morphological revision of the genus <em>Aiptasia</em> and the family Aiptasiidae (Cnidaria, Actiniaria, Metridioidea). Zootaxa, 3826(1), 55-100.</p>
<p>Rodr&iacute;guez, E. et al. (2014). Hidden among Sea Anemones: The First Comprehensive Phylogenetic Reconstruction of the Order Actiniaria (Cnidaria, Anthozoa, Hexacorallia) Reveals a Novel Group of Hexacorals. PLoS ONE 9(5): e96998.</p>
<hr /><p><strong>Characterizing the effects of age on the </strong><strong><em>Hydra</em></strong><strong> epigenome</strong></p>
<p>Jack Cazet1, Celina Juliano2, and Jodi Nunnari1</p>
<p>1Bay Area Institute of Science, Altos Labs, Redwood City, CA 94065</p>
<p>2Department of Molecular and Cellular Biology, University of California, Davis, CA 95616</p>
<p>Aging is associated with a decrease in the fidelity of cell-type-specific transcription, resulting in increased epigenetic and transcriptional heterogeneity within a given cell type through a process called epigenetic drift. In mammals, such regulatory deficiencies can be reversed through the activation of the embryonic pluripotency gene network (PGN) via transcription factors such as Sox2, Klf4, and Oct4. This rejuvenation is achieved through a dramatic reprograming event that resets the epigenome. Notably, the PGN in mammals does not appear to be well conserved outside of Vertebrata, and the regulation of cell potency and epigenetic reprogramming remains largely uncharacterized beyond a small number of well-studied bilaterian species.</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The freshwater cnidarian polyp <em>Hydra</em> does not appear to age, exhibiting constant mortality and fertility over time, but the basis for this lack of senescence is unclear. I propose to characterize the mechanisms by which <em>Hydra</em> escape the deleterious effects of epigenetic drift using the following experiments: 1) I will use ATAC-seq and CUT&amp;Tag to profile the epigenomes of young and old ISCs to determine if epigenetic heterogeneity increases with age; 2) I will characterize the transcriptional and epigenetic consequences of expressing a mutated version of HyLMN that is expected to cause disruptions in the nuclear lamina, a cellular phenotype that accelerates aging in mammals; and 3) I will characterize the epigenetic changes that occur as ISCs differentiate into germline stem cells to determine if ISCs undergo epigenetic reprogramming during germline specification in a manner similar to bilaterians. These experiments will help clarify whether <em>Hydra</em> escape aging through an exceptional ability to maintain epigenetic stability over time, or if <em>Hydra</em> are uniquely adapted to tolerate the downstream effects of any epigenetic drift that does occur.</p>

